# translation of gl.po to 
# translation of anaconda.po to 
# This file is distributed under the same license as the PACKAGE package.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: gl\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2007-03-16 15:11-0400\n"
"PO-Revision-Date: 2006-07-20 15:12+0200\n"
"Last-Translator: \n"
"Language-Team:  <gl@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: KBabel 1.9.1\n"

#: ../anaconda:262
msgid "Unknown Error"
msgstr "Erro Descoñecido"

#: ../anaconda:265
#, c-format
msgid "Error pulling second part of kickstart config: %s!"
msgstr ""

#: ../anaconda:279 ../cmdline.py:69 ../gui.py:957 ../text.py:400
#, c-format, python-format
msgid ""
"The following error was found while parsing your kickstart configuration:\n"
"\n"
"%s"
msgstr ""

#: ../anaconda:401
msgid "Press <enter> for a shell"
msgstr "Prema <intro> para obter unha shell"

#: ../anaconda:416 ../gui.py:235 ../rescue.py:46 ../rescue.py:247
#: ../rescue.py:321 ../rescue.py:348 ../rescue.py:358 ../rescue.py:439
#: ../rescue.py:445 ../text.py:440 ../text.py:599 ../vnc.py:144
#: ../textw/confirm_text.py:26 ../textw/confirm_text.py:54
#: ../textw/constants_text.py:36 ../textw/network_text.py:36
#: ../textw/network_text.py:46 ../textw/network_text.py:68
#: ../textw/network_text.py:74 ../textw/network_text.py:222
#: ../textw/network_text.py:803 ../textw/network_text.py:811
#: ../loader2/cdinstall.c:140 ../loader2/cdinstall.c:141
#: ../loader2/cdinstall.c:258 ../loader2/cdinstall.c:261
#: ../loader2/cdinstall.c:382 ../loader2/cdinstall.c:387
#: ../loader2/cdinstall.c:392 ../loader2/cdinstall.c:463
#: ../loader2/dirbrowser.c:139 ../loader2/driverdisk.c:273
#: ../loader2/driverdisk.c:304 ../loader2/driverdisk.c:338
#: ../loader2/driverdisk.c:376 ../loader2/driverdisk.c:390
#: ../loader2/driverdisk.c:404 ../loader2/driverdisk.c:414
#: ../loader2/driverdisk.c:581 ../loader2/driverdisk.c:618
#: ../loader2/driverselect.c:70 ../loader2/driverselect.c:207
#: ../loader2/hdinstall.c:106 ../loader2/hdinstall.c:157
#: ../loader2/hdinstall.c:320 ../loader2/hdinstall.c:371
#: ../loader2/hdinstall.c:406 ../loader2/hdinstall.c:476
#: ../loader2/hdinstall.c:519 ../loader2/hdinstall.c:532 ../loader2/kbd.c:131
#: ../loader2/kickstart.c:127 ../loader2/kickstart.c:137
#: ../loader2/kickstart.c:180 ../loader2/kickstart.c:279
#: ../loader2/kickstart.c:339 ../loader2/kickstart.c:485 ../loader2/lang.c:106
#: ../loader2/lang.c:369 ../loader2/loader.c:338 ../loader2/loader.c:351
#: ../loader2/loader.c:362 ../loader2/loader.c:838 ../loader2/loader.c:1002
#: ../loader2/mediacheck.c:329 ../loader2/mediacheck.c:387
#: ../loader2/mediacheck.c:432 ../loader2/method.c:156 ../loader2/method.c:374
#: ../loader2/method.c:459 ../loader2/modules.c:1042 ../loader2/modules.c:1055
#: ../loader2/net.c:268 ../loader2/net.c:307 ../loader2/net.c:739
#: ../loader2/net.c:1064 ../loader2/net.c:1611 ../loader2/net.c:1634
#: ../loader2/net.c:1820 ../loader2/nfsinstall.c:57
#: ../loader2/nfsinstall.c:125 ../loader2/nfsinstall.c:205
#: ../loader2/nfsinstall.c:213 ../loader2/nfsinstall.c:251
#: ../loader2/telnetd.c:87 ../loader2/urlinstall.c:79
#: ../loader2/urlinstall.c:151 ../loader2/urlinstall.c:164
#: ../loader2/urlinstall.c:456 ../loader2/urlinstall.c:465
#: ../loader2/urlinstall.c:476 ../loader2/urls.c:282 ../loader2/urls.c:347
#: ../loader2/urls.c:352 ../loader2/urls.c:358 ../loader2/urls.c:470
msgid "OK"
msgstr "Aceptar"

#: ../anaconda:423
msgid ""
"You do not have enough RAM to use the graphical installer.  Starting text "
"mode."
msgstr ""
"Non ten memoria dabondo para usar o instalador gráfico.  Iniciando o modo "
"texto."

#: ../anaconda:439
msgid "No video hardware found, assuming headless"
msgstr ""

#: ../anaconda:446 ../anaconda:898
msgid "Unable to instantiate a X hardware state object."
msgstr ""

#: ../anaconda:503
msgid "Starting graphical installation..."
msgstr "Iniciando a instalación gráfica..."

#: ../anaconda:763
msgid "Install class forcing text mode installation"
msgstr ""

#: ../anaconda:795
msgid "Graphical installation not available...  Starting text mode."
msgstr "Instalación gráfica non dispoñible...  Iniciando o modo texto."

#: ../anaconda:803
msgid "DISPLAY variable not set. Starting text mode!"
msgstr "A variable DISPLAY non está establecida. Iniciando o modo texto!"

#: ../anaconda:864
msgid "Unknown install method"
msgstr "Método de instalación descoñecido"

#: ../anaconda:865
msgid "You have specified an install method which isn't supported by anaconda."
msgstr ""
"Especificou un método de instalación que non está soportado por anaconda."

#: ../anaconda:867
#, c-format
msgid "unknown install method: %s"
msgstr "método de instalación descoñecido: %s"

#: ../autopart.py:955
#, python-format
msgid ""
"Could not allocate cylinder-based partitions as primary partitions.\n"
"\n"
"%s"
msgstr ""

#: ../autopart.py:960
#, fuzzy, python-format
msgid ""
"Could not allocate partitions as primary partitions.\n"
"\n"
"%s"
msgstr "Non se puideron asignar as particións"

#: ../autopart.py:965
#, fuzzy, python-format
msgid ""
"Could not allocate cylinder-based partitions.\n"
"\n"
"%s"
msgstr "Non se puideron asignar as particións"

#: ../autopart.py:1025
#, python-format
msgid ""
"Boot partition %s doesn't belong to a BSD disk label. SRM won't be able to "
"boot from this partition. Use a partition belonging to a BSD disk label or "
"change this device disk label to BSD."
msgstr ""

#: ../autopart.py:1027
#, python-format
msgid ""
"Boot partition %s doesn't belong to a disk with enough free space at its "
"beginning for the bootloader to live on. Make sure that there's at least 5MB "
"of free space at the beginning of the disk that contains /boot"
msgstr ""

#: ../autopart.py:1029
#, python-format
msgid ""
"Boot partition %s isn't a VFAT partition.  EFI won't be able to boot from "
"this partition."
msgstr ""

#: ../autopart.py:1031
msgid ""
"The boot partition must entirely be in the first 4GB of the disk.  "
"OpenFirmware won't be able to boot this installation."
msgstr ""

#: ../autopart.py:1038
#, python-format
msgid ""
"Boot partition %s may not meet booting constraints for your architecture."
msgstr ""

#: ../autopart.py:1064
#, python-format
msgid ""
"Adding this partition would not leave enough disk space for already "
"allocated logical volumes in %s."
msgstr ""

#: ../autopart.py:1259
msgid "Requested Partition Does Not Exist"
msgstr "A Partición Solicitada Non Existe"

#: ../autopart.py:1260
#, python-format
msgid ""
"Unable to locate partition %s to use for %s.\n"
"\n"
"Press 'OK' to reboot your system."
msgstr ""
"Non foi posible localizar a partición %s para usar para %s.\n"
"\n"
"Prema 'Aceptar' para reiniciar o sistema."

#: ../autopart.py:1287
msgid "Requested Raid Device Does Not Exist"
msgstr "O Dispositivo RAID Solicitado Non Existe"

#: ../autopart.py:1288
#, python-format
msgid ""
"Unable to locate raid device %s to use for %s.\n"
"\n"
"Press 'OK' to reboot your system."
msgstr ""
"Non foi posible localizar o dispositivo RAID %s para usar para %s.\n"
"\n"
"Prema 'Aceptar' para reiniciar o sistema."

#: ../autopart.py:1319
msgid "Requested Volume Group Does Not Exist"
msgstr ""

#: ../autopart.py:1320
#, python-format
msgid ""
"Unable to locate volume group %s to use for %s.\n"
"\n"
"Press 'OK' to reboot your system."
msgstr ""

#: ../autopart.py:1357
msgid "Requested Logical Volume Does Not Exist"
msgstr ""

#: ../autopart.py:1358
#, python-format
msgid ""
"Unable to locate logical volume %s to use for %s.\n"
"\n"
"Press 'OK' to reboot your system."
msgstr ""

#: ../autopart.py:1474 ../autopart.py:1521
msgid "Automatic Partitioning Errors"
msgstr "Erros do Particionamento Automático"

#: ../autopart.py:1475
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"Press 'OK' to reboot your system."
msgstr ""

#: ../autopart.py:1485
msgid "Warnings During Automatic Partitioning"
msgstr "Advertencias durante o Particionamento Automático"

#: ../autopart.py:1486
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"Ocorreron as seguintes advertencias durante o particionamento automático:\n"
"\n"
"%s"

#: ../autopart.py:1500 ../autopart.py:1517
msgid ""
"\n"
"\n"
"Press 'OK' to reboot your system."
msgstr ""
"\n"
"\n"
"Prema 'Aceptar' para reiniciar o sistema."

#: ../autopart.py:1501 ../iw/partition_gui.py:997
#: ../textw/partition_text.py:228
#, fuzzy
msgid "Error Partitioning"
msgstr "Erro ó Particionar"

#: ../autopart.py:1502
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%s.%s"
msgstr ""

#: ../autopart.py:1519
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""

#: ../autopart.py:1522
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %s"
msgstr ""

#: ../autopart.py:1533
msgid "Unrecoverable Error"
msgstr ""

#: ../autopart.py:1534
msgid "Your system will now be rebooted."
msgstr ""

#: ../autopart.py:1681 ../bootloader.py:197 ../image.py:435
#: ../partedUtils.py:304 ../partedUtils.py:334 ../partedUtils.py:1037
#: ../partedUtils.py:1102 ../upgrade.py:348 ../yuminstall.py:1084
#: ../iw/blpasswidget.py:145 ../iw/bootloader_advanced_gui.py:41
#: ../iw/task_gui.py:89 ../iw/upgrade_swap_gui.py:188
#: ../iw/upgrade_swap_gui.py:196 ../iw/upgrade_swap_gui.py:203
#: ../textw/bootloader_text.py:123 ../textw/bootloader_text.py:448
#: ../textw/partition_text.py:232 ../textw/upgrade_text.py:172
#: ../loader2/loader.c:405
msgid "Warning"
msgstr "Advertencia"

#: ../autopart.py:1687
msgid ""
"Automatic Partitioning sets partitions based on the selected installation "
"type. You also can customize the partitions once they have been created.\n"
"\n"
"The manual disk partitioning tool, Disk Druid, allows you to create "
"partitions in an interactive environment. You can set the file system types, "
"mount points, partition sizes, and more."
msgstr ""

#: ../autopart.py:1698
msgid ""
"Before automatic partitioning can be set up by the installation program, you "
"must choose how to use the space on your hard drives."
msgstr ""

#: ../autopart.py:1703
msgid "Remove all partitions on this system"
msgstr "Eliminar tódalas particións deste sistema"

#: ../autopart.py:1704
msgid "Remove all Linux partitions on this system"
msgstr "Eliminar tódalas particións Linux deste sistema"

#: ../autopart.py:1705
msgid "Keep all partitions and use existing free space"
msgstr ""

#: ../autopart.py:1707
#, python-format
msgid ""
"You have chosen to remove all partitions (ALL DATA) on the following drives:%"
"s\n"
"Are you sure you want to do this?"
msgstr ""

#: ../autopart.py:1711
#, python-format
msgid ""
"You have chosen to remove all Linux partitions (and ALL DATA on them) on the "
"following drives:%s\n"
"Are you sure you want to do this?"
msgstr ""

#: ../backend.py:164
#, fuzzy, python-format
msgid "Upgrading %s\n"
msgstr "Actualizando %s-%s-%s.%s.\n"

#: ../backend.py:166
#, fuzzy, python-format
msgid "Installing %s\n"
msgstr "_Instalar %s"

#: ../bootloader.py:129
msgid "Bootloader"
msgstr "Cargador de arrinque"

#: ../bootloader.py:129
msgid "Installing bootloader..."
msgstr "Instalando o cargador de arrinque..."

#: ../bootloader.py:198
msgid ""
"No kernel packages were installed on your system.  Your boot loader "
"configuration will not be changed."
msgstr ""

#: ../cmdline.py:44
msgid "Completed"
msgstr "Completado"

#: ../cmdline.py:52
#, fuzzy
msgid "In progress...   "
msgstr "En proceso...   "

#: ../cmdline.py:81
msgid "Can't have a question in command line mode!"
msgstr ""

#: ../cmdline.py:100
msgid "Parted exceptions can't be handled in command line mode!"
msgstr ""

#: ../constants.py:65
#, python-format
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report against anaconda "
"at %s"
msgstr ""

#: ../exception.py:414 ../exception.py:431
msgid "Dump Written"
msgstr ""

#: ../exception.py:415
msgid ""
"Your system's state has been successfully written to the floppy. Your system "
"will now be rebooted."
msgstr ""

#: ../exception.py:418 ../exception.py:435 ../fsset.py:1766 ../fsset.py:2456
#: ../gui.py:962 ../gui.py:1114 ../harddrive.py:80 ../image.py:93
#: ../image.py:445 ../image.py:519 ../packages.py:340 ../iw/confirm_gui.py:32
#: ../textw/confirm_text.py:38 ../textw/confirm_text.py:66
msgid "_Reboot"
msgstr "_Reiniciar"

#: ../exception.py:423 ../exception.py:440
msgid "Dump Not Written"
msgstr ""

#: ../exception.py:424
msgid "There was a problem writing the system state to the floppy."
msgstr "Houbo un problema ó escribir o estado do sistema no disquete."

#: ../exception.py:432
msgid ""
"Your system's state has been successfully written to the remote host.  Your "
"system will now be rebooted."
msgstr ""

#: ../exception.py:441
msgid "There was a problem writing the system state to the remote host."
msgstr ""

#: ../fsset.py:214
msgid "Checking for Bad Blocks"
msgstr ""

#: ../fsset.py:215
#, python-format
msgid "Checking for bad blocks on /dev/%s..."
msgstr ""

#: ../fsset.py:661 ../fsset.py:1450 ../fsset.py:1481 ../fsset.py:1542
#: ../fsset.py:1553 ../fsset.py:1607 ../fsset.py:1618 ../fsset.py:1658
#: ../fsset.py:1708 ../fsset.py:1774 ../fsset.py:1793 ../image.py:135
#: ../image.py:176 ../image.py:308 ../partIntfHelpers.py:406
#: ../urlinstall.py:110 ../urlinstall.py:208 ../yuminstall.py:605
#: ../yuminstall.py:722 ../yuminstall.py:738 ../yuminstall.py:923
#: ../iw/autopart_type.py:201 ../iw/netconfig_dialog.py:213
#: ../iw/osbootwidget.py:216 ../iw/osbootwidget.py:225
#: ../iw/raid_dialog_gui.py:618 ../iw/raid_dialog_gui.py:657
#: ../iw/task_gui.py:79 ../iw/task_gui.py:145 ../textw/grpselect_text.py:116
#: ../textw/partition_text.py:1670 ../textw/partition_text.py:1676
#: ../textw/partition_text.py:1698 ../textw/upgrade_text.py:160
#: ../textw/upgrade_text.py:167 ../loader2/cdinstall.c:141
#: ../loader2/cdinstall.c:463 ../loader2/driverdisk.c:338
#: ../loader2/driverdisk.c:376 ../loader2/driverdisk.c:404
#: ../loader2/driverdisk.c:414 ../loader2/driverdisk.c:479
#: ../loader2/hdinstall.c:106 ../loader2/hdinstall.c:157
#: ../loader2/hdinstall.c:371 ../loader2/hdinstall.c:476
#: ../loader2/hdinstall.c:519 ../loader2/hdinstall.c:532
#: ../loader2/kickstart.c:279 ../loader2/lang.c:106 ../loader2/loader.c:362
#: ../loader2/loader.c:838 ../loader2/mediacheck.c:329
#: ../loader2/mediacheck.c:387 ../loader2/method.c:156 ../loader2/method.c:374
#: ../loader2/method.c:459 ../loader2/nfsinstall.c:125
#: ../loader2/nfsinstall.c:205 ../loader2/nfsinstall.c:213
#: ../loader2/telnetd.c:87 ../loader2/urlinstall.c:79
#: ../loader2/urlinstall.c:151 ../loader2/urlinstall.c:164
#: ../loader2/urls.c:347 ../loader2/urls.c:352
msgid "Error"
msgstr "Erro"

#: ../fsset.py:662
#, python-format
msgid ""
"An error occurred migrating %s to ext3.  It is possible to continue without "
"migrating this file system if desired.\n"
"\n"
"Would you like to continue without migrating %s?"
msgstr ""

#: ../fsset.py:1357
msgid "RAID Device"
msgstr "Dispositivo RAID"

#: ../fsset.py:1361 ../fsset.py:1367
msgid "Apple Bootstrap"
msgstr ""

#: ../fsset.py:1372 ../partitions.py:900
msgid "PPC PReP Boot"
msgstr ""

#: ../fsset.py:1375
msgid "First sector of boot partition"
msgstr "Primeiro sector da partición de arrinque"

#: ../fsset.py:1376
msgid "Master Boot Record (MBR)"
msgstr ""

#: ../fsset.py:1451
#, python-format
msgid ""
"An error occurred trying to initialize swap on device %s.  This problem is "
"serious, and the install cannot continue.\n"
"\n"
"Press <Enter> to reboot your system."
msgstr ""

#: ../fsset.py:1480 ../packages.py:283 ../rescue.py:287 ../rescue.py:289
#: ../textw/upgrade_text.py:118 ../loader2/cdinstall.c:258
#: ../loader2/cdinstall.c:261 ../loader2/method.c:421
msgid "Skip"
msgstr ""

#: ../fsset.py:1480 ../upgrade.py:59 ../textw/complete_text.py:41
msgid "Reboot"
msgstr "Reiniciar"

#: ../fsset.py:1501
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     /dev/%s\n"
"\n"
"is a version 0 Linux swap partition. If you want to use this device, you "
"must reformat as a version 1 Linux swap partition. If you skip it, the "
"installer will ignore it during the installation."
msgstr ""

#: ../fsset.py:1508
msgid "Reformat"
msgstr ""

#: ../fsset.py:1512
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     /dev/%s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend partition, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""

#: ../fsset.py:1520
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     /dev/%s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend partition, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap partitions."
msgstr ""

#: ../fsset.py:1530
msgid ""
"\n"
"\n"
"Choose Skip if you want the installer to ignore this partition during the "
"upgrade.  Choose Format to reformat the partition as swap space.  Choose "
"Reboot to restart the system."
msgstr ""

#: ../fsset.py:1536 ../iw/partition_gui.py:368
msgid "Format"
msgstr "Formatar"

#: ../fsset.py:1543
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"partition.\n"
"\n"
"Press OK to reboot your system."
msgstr ""

#: ../fsset.py:1554
#, python-format
msgid ""
"Error enabling swap device %s: %s\n"
"\n"
"This most likely means this swap partition has not been initialized.\n"
"\n"
"Press OK to reboot your system."
msgstr ""

#: ../fsset.py:1608
#, python-format
msgid ""
"Bad blocks have been detected on device /dev/%s. We do not recommend you use "
"this device.\n"
"\n"
"Press <Enter> to reboot your system"
msgstr ""

#: ../fsset.py:1619
#, python-format
msgid ""
"An error occurred searching for bad blocks on %s.  This problem is serious, "
"and the install cannot continue.\n"
"\n"
"Press <Enter> to reboot your system."
msgstr ""

#: ../fsset.py:1659
#, python-format
msgid ""
"An error occurred trying to format %s.  This problem is serious, and the "
"install cannot continue.\n"
"\n"
"Press <Enter> to reboot your system."
msgstr ""

#: ../fsset.py:1709
#, python-format
msgid ""
"An error occurred trying to migrate %s.  This problem is serious, and the "
"install cannot continue.\n"
"\n"
"Press <Enter> to reboot your system."
msgstr ""

#: ../fsset.py:1735 ../fsset.py:1744
msgid "Invalid mount point"
msgstr "Punto de montaxe incorrecto"

#: ../fsset.py:1736
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to reboot your system."
msgstr ""

#: ../fsset.py:1745
#, python-format
msgid ""
"An error occurred when trying to create %s: %s.  This is a fatal error and "
"the install cannot continue.\n"
"\n"
"Press <Enter> to reboot your system."
msgstr ""

#: ../fsset.py:1758
#, fuzzy
msgid "Unable to mount filesystem"
msgstr "Non Foi Posible Eliminar"

#: ../fsset.py:1759
#, python-format
msgid ""
"An error occurred mounting device %s as %s.  You may continue installation, "
"but there may be problems."
msgstr ""

#: ../fsset.py:1767 ../image.py:93 ../image.py:446 ../kickstart.py:987
#: ../kickstart.py:1025 ../yuminstall.py:718 ../iw/partition_gui.py:1011
msgid "_Continue"
msgstr "_Continuar"

#: ../fsset.py:1775
#, python-format
msgid ""
"Error mounting device %s as %s: %s\n"
"\n"
"This most likely means this partition has not been formatted.\n"
"\n"
"Press OK to reboot your system."
msgstr ""

#: ../fsset.py:1794
msgid ""
"Error finding / entry.\n"
"\n"
"This is most likely means that your fstab is incorrect.\n"
"\n"
"Press OK to reboot your system."
msgstr ""

#: ../fsset.py:2448
msgid "Duplicate Labels"
msgstr ""

#: ../fsset.py:2449
#, python-format
msgid ""
"Multiple devices on your system are labelled %s.  Labels across devices must "
"be unique for your system to function properly.\n"
"\n"
"Please fix this problem and restart the installation process."
msgstr ""

#: ../fsset.py:2709
msgid "Formatting"
msgstr "Formatando"

#: ../fsset.py:2710
#, python-format
msgid "Formatting %s file system..."
msgstr "Formatando o sistema de ficheiros %s..."

#: ../gui.py:107
msgid "An error occurred copying the screenshots over."
msgstr ""

#: ../gui.py:118
msgid "Screenshots Copied"
msgstr ""

#: ../gui.py:119
msgid ""
"The screenshots have been saved into the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""

#: ../gui.py:162
msgid "Saving Screenshot"
msgstr "Gardando Captura de Pantalla"

#: ../gui.py:163
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr ""

#: ../gui.py:166
msgid "Error Saving Screenshot"
msgstr "Erro ó gardar Captura de Pantalla"

#: ../gui.py:167
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""

#: ../gui.py:232 ../text.py:437
msgid "Fix"
msgstr ""

#: ../gui.py:233 ../rescue.py:203 ../text.py:438 ../upgrade.py:59
#: ../textw/bootloader_text.py:68 ../textw/constants_text.py:48
#: ../loader2/driverdisk.c:520 ../loader2/driverdisk.c:531
#: ../loader2/hdinstall.c:265 ../loader2/loader.c:405
msgid "Yes"
msgstr "Sí"

#: ../gui.py:234 ../rescue.py:203 ../rescue.py:205 ../text.py:439
#: ../textw/bootloader_text.py:68 ../textw/constants_text.py:52
#: ../loader2/driverdisk.c:520 ../loader2/driverdisk.c:531
#: ../loader2/loader.c:405
msgid "No"
msgstr "Non"

#: ../gui.py:236 ../text.py:441 ../text.py:569 ../loader2/net.c:93
#: ../loader2/net.c:325 ../loader2/net.c:583 ../loader2/net.c:691
#: ../loader2/net.c:791 ../loader2/net.c:799 ../loader2/net.c:1206
#: ../loader2/net.c:1212
msgid "Retry"
msgstr ""

#: ../gui.py:237 ../text.py:442
msgid "Ignore"
msgstr "Ignorar"

#: ../gui.py:238 ../gui.py:777 ../partIntfHelpers.py:237
#: ../partIntfHelpers.py:529 ../text.py:103 ../text.py:104 ../text.py:414
#: ../text.py:416 ../text.py:443 ../iw/bootloader_advanced_gui.py:47
#: ../textw/bootloader_text.py:198 ../textw/constants_text.py:40
#: ../loader2/dirbrowser.c:139 ../loader2/driverdisk.c:274
#: ../loader2/kickstart.c:339 ../loader2/loader.c:351
msgid "Cancel"
msgstr "Cancelar"

#: ../gui.py:604 ../text.py:357
#, fuzzy
msgid "Installation Key"
msgstr "Progreso da Instalación"

#: ../gui.py:787 tmp/anaconda.glade.h:2
msgid "_Debug"
msgstr ""

#: ../gui.py:959 ../text.py:402
msgid "Error Parsing Kickstart Config"
msgstr ""

#: ../gui.py:967 ../text.py:412
msgid ""
"Please insert a floppy now. All contents of the disk will be erased, so "
"please choose your diskette carefully."
msgstr ""

#: ../gui.py:1014
msgid "default:LTR"
msgstr ""

#: ../gui.py:1100 ../text.py:564
msgid "Error!"
msgstr "Erro!"

#: ../gui.py:1101 ../text.py:565
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""

#: ../gui.py:1106 ../gui.py:1111 ../packages.py:332 ../packages.py:337
msgid "_Exit"
msgstr "_Saír"

#: ../gui.py:1107 ../yuminstall.py:608
msgid "_Retry"
msgstr ""

#: ../gui.py:1110 ../packages.py:336
msgid "The installer will now exit..."
msgstr ""

#: ../gui.py:1113 ../packages.py:339
msgid "Your system will now be rebooted..."
msgstr ""

#: ../gui.py:1116 ../packages.py:341
msgid "Rebooting System"
msgstr "Reiniciando o Sistema"

#: ../gui.py:1180
#, python-format
msgid "%s Installer"
msgstr "Instalador de %s"

#: ../gui.py:1187
msgid "Unable to load title bar"
msgstr "Non foi posible cargar a barra de título"

#: ../gui.py:1242
msgid "Install Window"
msgstr ""

#: ../harddrive.py:44 ../image.py:146 ../image.py:476
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file or perhaps a "
"corrupt package.  Please verify your installation images and that you have "
"all the required media.\n"
"\n"
"If you reboot, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: ../harddrive.py:71 ../image.py:509
msgid "Missing ISO 9660 Image"
msgstr ""

#: ../harddrive.py:72
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry. Click Reboot  to abort "
"the installation."
msgstr ""

#: ../harddrive.py:81 ../image.py:520
msgid "Re_try"
msgstr ""

#: ../image.py:84
msgid "Required Install Media"
msgstr ""

#: ../image.py:85
#, python-format
msgid ""
"The software you have selected to install will require the following CDs:\n"
"\n"
"%s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and reboot please select \"Reboot\"."
msgstr ""

#: ../image.py:93 ../packages.py:286 ../packages.py:337 ../packages.py:340
#: ../yuminstall.py:930 ../iw/confirm_gui.py:32 ../textw/confirm_text.py:38
#: ../textw/confirm_text.py:66
msgid "_Back"
msgstr "_Atrás"

#: ../image.py:136 ../urlinstall.py:209
#, python-format
msgid ""
"An error occurred unmounting the CD.  Please make sure you're not accessing %"
"s from the shell on tty2 and then click OK to retry."
msgstr ""

#: ../image.py:166 ../urlinstall.py:100
msgid "Copying File"
msgstr "Copiando Ficheiro"

#: ../image.py:167 ../urlinstall.py:101
msgid "Transferring install image to hard drive..."
msgstr "Transferindo a imaxe de instalación ó disco duro..."

#: ../image.py:177 ../urlinstall.py:111
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""

#: ../image.py:266
msgid "Change CDROM"
msgstr "Cambiar CDROM"

#: ../image.py:267
#, python-format
msgid "Please insert %s disc %d to continue."
msgstr "Insira o disco %s %d para continuar."

#: ../image.py:302
msgid "Wrong CDROM"
msgstr "CDROM incorrecto"

#: ../image.py:303
#, python-format
msgid "That's not the correct %s CDROM."
msgstr "Este non é o CDROM de %s correcto."

#: ../image.py:309
msgid "Unable to access the CDROM."
msgstr "Non foi posible acceder ó CDROM"

#: ../image.py:362
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file or perhaps a "
"corrupt package.  Please verify your installation tree contains all required "
"packages.\n"
"\n"
"If you reboot, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: ../image.py:436
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you reboot and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""

#: ../image.py:510
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the "
"server.\n"
"\n"
"Please copy this image to the remote server's share path and click Retry. "
"Click Reboot to abort the installation."
msgstr ""

#: ../installclass.py:66
msgid "Install on System"
msgstr ""

#: ../iscsi.py:197 ../iscsi.py:198
#, fuzzy
msgid "Initializing iSCSI initiator"
msgstr "Inicializando"

#: ../kickstart.py:75
msgid "Scriptlet Failure"
msgstr ""

#: ../kickstart.py:76
#, python-format
msgid ""
"There was an error running the scriptlet.  You may examine the output in %"
"s.  This is a fatal error and your install will be aborted.\n"
"\n"
"Press the OK button to reboot your system."
msgstr ""

#: ../kickstart.py:930 ../kickstart.py:947
msgid "Running..."
msgstr "Executando..."

#: ../kickstart.py:931
msgid "Running post-install scripts"
msgstr ""

#: ../kickstart.py:948
msgid "Running pre-install scripts"
msgstr ""

#: ../kickstart.py:979
msgid "Missing Package"
msgstr ""

#: ../kickstart.py:980
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort your installation?"
msgstr ""

#: ../kickstart.py:986 ../kickstart.py:1024 ../yuminstall.py:718
#: ../yuminstall.py:720
msgid "_Abort"
msgstr "_Abortar"

#: ../kickstart.py:1016
msgid "Missing Group"
msgstr ""

#: ../kickstart.py:1017
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort your installation?"
msgstr ""

#: ../livecd.py:77 ../livecd.py:263
#, fuzzy
msgid "Unable to find image"
msgstr "Non foi posible cargar a barra de título"

#: ../livecd.py:78 ../livecd.py:264
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""

#: ../livecd.py:82 ../livecd.py:268
#, fuzzy
msgid "Exit installer"
msgstr "Instalador de %s"

#: ../livecd.py:113
#, fuzzy
msgid "Copying live image to hard drive."
msgstr "Transferindo a imaxe de instalación ó disco duro..."

#: ../livecd.py:145
msgid "Doing post-installation"
msgstr ""

#: ../livecd.py:146
#, fuzzy
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes..."
msgstr "Iniciando o proceso de instalación.  Isto pode levar varios minutos..."

#: ../network.py:51
msgid "Hostname must be 64 or less characters in length."
msgstr "O nome de máquina de ter 64 caracteres ou menos de lonxitude."

#: ../network.py:54
msgid "Hostname must start with a valid character in the range 'a-z' or 'A-Z'"
msgstr ""

#: ../network.py:59
msgid "Hostnames can only contain the characters 'a-z', 'A-Z', '-', or '.'"
msgstr ""

#: ../network.py:89
msgid "IP address is missing."
msgstr ""

#: ../network.py:93
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""

#: ../network.py:96
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr ""

#: ../network.py:98
#, python-format
msgid "'%s' is an invalid IP address."
msgstr ""

#: ../packages.py:257
msgid "Invalid Key"
msgstr ""

#: ../packages.py:258
msgid "The key you entered is invalid."
msgstr ""

#: ../packages.py:286
msgid "_Skip"
msgstr ""

#: ../packages.py:318
msgid "Warning! This is pre-release software!"
msgstr ""

#: ../packages.py:319
#, python-format
msgid ""
"Thank you for downloading this pre-release of %s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %s\n"
"\n"
"and file a report against '%s'.\n"
msgstr ""

#: ../packages.py:332
msgid "_Install anyway"
msgstr "_Instalar de tódolos xeitos"

#: ../partIntfHelpers.py:35
msgid "Please enter a volume group name."
msgstr ""

#: ../partIntfHelpers.py:39
msgid "Volume Group Names must be less than 128 characters"
msgstr ""

#: ../partIntfHelpers.py:42
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr ""

#: ../partIntfHelpers.py:47
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: ../partIntfHelpers.py:57
msgid "Please enter a logical volume name."
msgstr ""

#: ../partIntfHelpers.py:61
msgid "Logical Volume Names must be less than 128 characters"
msgstr ""

#: ../partIntfHelpers.py:65
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr ""

#: ../partIntfHelpers.py:71
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: ../partIntfHelpers.py:95
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""

#: ../partIntfHelpers.py:102
msgid "Please specify a mount point for this partition."
msgstr "Especifique un punto de montaxe para esta partición."

#: ../partIntfHelpers.py:112
#, python-format
msgid "This partition is part of the RAID device /dev/md%s."
msgstr "Esta partición é parte do dispositivo RAID /dev/md%s."

#: ../partIntfHelpers.py:115
msgid "This partition is part of a RAID device."
msgstr "Esta partición é parte dun dispositivo RAID."

#: ../partIntfHelpers.py:120
#, python-format
msgid "This partition is part of the LVM volume group '%s'."
msgstr ""

#: ../partIntfHelpers.py:123
msgid "This partition is part of a LVM volume group."
msgstr ""

#: ../partIntfHelpers.py:138 ../partIntfHelpers.py:146
#: ../partIntfHelpers.py:153 ../partIntfHelpers.py:163
#: ../partIntfHelpers.py:187
msgid "Unable To Delete"
msgstr "Non Foi Posible Eliminar"

#: ../partIntfHelpers.py:139
msgid "You must first select a partition to delete."
msgstr "Debe seleccionar primeiro unha partición para poder eliminar."

#: ../partIntfHelpers.py:147
msgid "You cannot delete free space."
msgstr "Non pode eliminar o espacio baleiro."

#: ../partIntfHelpers.py:154
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr ""

#: ../partIntfHelpers.py:164
#, python-format
msgid ""
"You cannot delete this partition, as it is an extended partition which "
"contains %s"
msgstr ""
"Non pode eliminar esta partición, xa que é unha partición extendida que "
"contén %s"

#: ../partIntfHelpers.py:182 ../iw/raid_dialog_gui.py:554
msgid "This partition is holding the data for the hard drive install."
msgstr ""

#: ../partIntfHelpers.py:188
msgid ""
"You cannot delete this partition:\n"
"\n"
msgstr ""
"Non pode eliminar esta partición:\n"
"\n"

#: ../partIntfHelpers.py:233 ../partIntfHelpers.py:528
#: ../iw/lvm_dialog_gui.py:755
msgid "Confirm Delete"
msgstr "Confirmar A Eliminación"

#: ../partIntfHelpers.py:234
#, python-format
msgid "You are about to delete all partitions on the device '/dev/%s'."
msgstr "Está a piques de eliminar tódalas particións do dispositivo '/dev/%s'."

#: ../partIntfHelpers.py:237 ../partIntfHelpers.py:529
#: ../iw/lvm_dialog_gui.py:758 ../iw/lvm_dialog_gui.py:1107
#: ../iw/osbootwidget.py:104 ../iw/partition_gui.py:1353
msgid "_Delete"
msgstr "_Eliminar"

#: ../partIntfHelpers.py:295
msgid "Notice"
msgstr "Aviso"

#: ../partIntfHelpers.py:296
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"Non se van eliminar as seguintes particións porque se están usando:\n"
"\n"
"%s"

#: ../partIntfHelpers.py:312 ../partIntfHelpers.py:325
#: ../partIntfHelpers.py:351 ../partIntfHelpers.py:362
msgid "Unable To Edit"
msgstr ""

#: ../partIntfHelpers.py:313
msgid "You must select a partition to edit"
msgstr "Debe seleccionar unha partición para poder editar"

#: ../partIntfHelpers.py:325 ../partIntfHelpers.py:363
msgid ""
"You cannot edit this partition:\n"
"\n"
msgstr ""
"Non pode editar esta partición:\n"
"\n"

#: ../partIntfHelpers.py:352
#, python-format
msgid ""
"You cannot edit this partition, as it is an extended partition which "
"contains %s"
msgstr ""
"Non pode editar esta partición, xa que é unha partición extendida que contén "
"%s"

#: ../partIntfHelpers.py:384
msgid "Format as Swap?"
msgstr ""

#: ../partIntfHelpers.py:385
#, python-format
msgid ""
"/dev/%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""

#: ../partIntfHelpers.py:405
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""

#: ../partIntfHelpers.py:410
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then  "
"continue without formatting this partition."
msgstr ""

#: ../partIntfHelpers.py:418
msgid "Format?"
msgstr "Formatar?"

#: ../partIntfHelpers.py:418 ../iw/partition_gui.py:1009
msgid "_Modify Partition"
msgstr "_Modificar Partición"

#: ../partIntfHelpers.py:418
msgid "Do _Not Format"
msgstr "_Non Formatar"

#: ../partIntfHelpers.py:426
msgid "Error with Partitioning"
msgstr ""

#: ../partIntfHelpers.py:427
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of %s.\n"
"\n"
"%s"
msgstr ""

#: ../partIntfHelpers.py:441
msgid "Partitioning Warning"
msgstr "Advertencia de Particionamento"

#: ../partIntfHelpers.py:442
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: ../partIntfHelpers.py:456 ../iw/partition_gui.py:664
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""

#: ../partIntfHelpers.py:459
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""

#: ../partIntfHelpers.py:465
msgid "Format Warning"
msgstr ""

#: ../partIntfHelpers.py:513
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""

#: ../partIntfHelpers.py:517
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr ""

#: ../partIntfHelpers.py:520
msgid "You are about to delete a RAID device."
msgstr "Está a piques de eliminar un dispositivo RAID."

#: ../partIntfHelpers.py:523
#, python-format
msgid "You are about to delete the /dev/%s partition."
msgstr "Está a piques de eliminar a partición /dev/%s."

#: ../partIntfHelpers.py:526
msgid "The partition you selected will be deleted."
msgstr "Vaise eliminar a partición que seleccionou."

#: ../partIntfHelpers.py:536
msgid "Confirm Reset"
msgstr ""

#: ../partIntfHelpers.py:537
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""

#: ../partRequests.py:249
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: ../partRequests.py:252
#, python-format
msgid ""
"The mount point %s cannot be used.  It must be a symbolic link for proper "
"system operation.  Please select a different mount point."
msgstr ""

#: ../partRequests.py:259
msgid "This mount point must be on a linux file system."
msgstr ""

#: ../partRequests.py:280
#, python-format
msgid ""
"The mount point \"%s\" is already in use, please choose a different mount "
"point."
msgstr ""

#: ../partRequests.py:294
#, python-format
msgid ""
"The size of the %s partition (%10.2f MB) exceeds the maximum size of %10.2f "
"MB."
msgstr ""

#: ../partRequests.py:490
#, python-format
msgid ""
"The size of the requested partition (size = %s MB) exceeds the maximum size "
"of %s MB."
msgstr ""

#: ../partRequests.py:495
#, python-format
msgid "The size of the requested partition is negative! (size = %s MB)"
msgstr ""

#: ../partRequests.py:499
msgid "Partitions can't start below the first cylinder."
msgstr "As particións non poden comezar antes do primeiro cilindro."

#: ../partRequests.py:502
msgid "Partitions can't end on a negative cylinder."
msgstr "As particións non poden rematar nun cilindro negativo."

#: ../partRequests.py:671
msgid "No members in RAID request, or not RAID level specified."
msgstr ""

#: ../partRequests.py:679 ../partitions.py:954
msgid "Bootable partitions can only be on RAID1 devices."
msgstr ""

#: ../partRequests.py:683
#, python-format
msgid "A RAID device of type %s requires at least %s members."
msgstr ""

#: ../partRequests.py:692
#, python-format
msgid ""
"This RAID device can have a maximum of %s spares. To have more spares you "
"will need to add members to the RAID device."
msgstr ""

#: ../partRequests.py:926
msgid ""
"Logical volume size must be larger than the volume group's physical extent "
"size."
msgstr ""

#: ../partedUtils.py:205 ../textw/partition_text.py:559
msgid "Foreign"
msgstr ""

#: ../partedUtils.py:305
#, python-format
msgid ""
"The device %s is LDL formatted instead of CDL formatted.  LDL formatted "
"DASDs are not supported for usage during an install of %s.  If you wish to "
"use this disk for installation, it must be re-initialized causing the loss "
"of ALL DATA on this drive.\n"
"\n"
"Would you like to reformat this DASD using CDL format?"
msgstr ""

#: ../partedUtils.py:335
#, python-format
msgid ""
"/dev/%s currently has a %s partition layout.  To use this drive for the "
"installation of %s, it must be re-initialized, causing the loss of ALL DATA "
"on this drive.\n"
"\n"
"Would you like to re-initialize this drive?"
msgstr ""

#: ../partedUtils.py:344
msgid "_Ignore drive"
msgstr "_Ignorar unidade"

#: ../partedUtils.py:345
msgid "_Re-initialize drive"
msgstr ""

#: ../partedUtils.py:917
msgid "Initializing"
msgstr "Inicializando"

#: ../partedUtils.py:918
#, python-format
msgid "Please wait while formatting drive %s...\n"
msgstr "Agarde mentres se formata a unidade %s...\n"

#: ../partedUtils.py:1038
#, python-format
msgid ""
"The partition table on device %s (%s) was unreadable. To create new "
"partitions it must be initialized, causing the loss of ALL DATA on this "
"drive.\n"
"\n"
"This operation will override any previous installation choices about which "
"drives to ignore.\n"
"\n"
"Would you like to initialize this drive, erasing ALL DATA?"
msgstr ""

#: ../partedUtils.py:1103
#, python-format
msgid ""
"The partition table on device %s was unreadable. To create new partitions it "
"must be initialized, causing the loss of ALL DATA on this drive.\n"
"\n"
"This operation will override any previous installation choices about which "
"drives to ignore.\n"
"\n"
"Would you like to initialize this drive, erasing ALL DATA?"
msgstr ""

#: ../partedUtils.py:1224
msgid "No Drives Found"
msgstr "Non Se Atoparon Unidades"

#: ../partedUtils.py:1225
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""

#: ../partitioning.py:61
msgid "Installation cannot continue."
msgstr "A instalación non pode continuar."

#: ../partitioning.py:62
msgid ""
"The partitioning options you have chosen have already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""

#: ../partitioning.py:92
msgid "Low Memory"
msgstr ""

#: ../partitioning.py:93
msgid ""
"As you don't have much memory in this machine, we need to turn on swap space "
"immediately. To do this we'll have to write your new partition table to the "
"disk immediately. Is that OK?"
msgstr ""

#: ../partitions.py:816
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: ../partitions.py:821
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"A partición raíz ten menos de 250 megabytes o cal é normalmente moi pouco "
"para instalar %s."

#: ../partitions.py:840
msgid ""
"Your boot partition isn't on one of the first four partitions and thus won't "
"be bootable."
msgstr ""

#: ../partitions.py:847
msgid ""
"You must create a /boot/efi partition of type FAT and a size of 50 megabytes."
msgstr ""
"Debe crear unha partición /boot/efi de tipo FAT e cun tamaño de 50 megabytes."

#: ../partitions.py:867
msgid "You must create an Apple Bootstrap partition."
msgstr ""

#: ../partitions.py:889
msgid "You must create a PPC PReP Boot partition."
msgstr ""

#: ../partitions.py:897 ../partitions.py:908
#, python-format
msgid ""
"Your %s partition is less than %s megabytes which is lower than recommended "
"for a normal %s install."
msgstr ""

#: ../partitions.py:942
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""

#: ../partitions.py:945
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""

#: ../partitions.py:961
msgid "Bootable partitions cannot be on a logical volume."
msgstr ""

#: ../partitions.py:986
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: ../partitions.py:993
#, python-format
msgid ""
"You have specified more than 32 swap devices.  The kernel for %s only "
"supports 32 swap devices."
msgstr ""

#: ../partitions.py:1004
#, python-format
msgid ""
"You have allocated less swap space (%dM) than available RAM (%dM) on your "
"system.  This could negatively impact performance."
msgstr ""

#: ../partitions.py:1304
msgid "the partition in use by the installer."
msgstr ""

#: ../partitions.py:1307
msgid "a partition which is a member of a RAID array."
msgstr ""

#: ../partitions.py:1310
msgid "a partition which is a member of a LVM Volume Group."
msgstr ""

#: ../rescue.py:129
msgid "Starting Interface"
msgstr "Iniciando a Interface"

#: ../rescue.py:130
#, python-format
msgid "Attempting to start %s"
msgstr "Tentando iniciar %s"

#: ../rescue.py:175
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: ../rescue.py:201
msgid "Setup Networking"
msgstr "Configuración de Rede"

#: ../rescue.py:202
msgid "Do you want to start the network interfaces on this system?"
msgstr ""

#: ../rescue.py:243 ../text.py:595
msgid "Cancelled"
msgstr ""

#: ../rescue.py:244 ../text.py:596
msgid "I can't go to the previous step from here. You will have to try again."
msgstr ""

#: ../rescue.py:276 ../rescue.py:343 ../rescue.py:351 ../rescue.py:434
msgid "Rescue"
msgstr "Rescate"

#: ../rescue.py:277
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: ../rescue.py:287 ../iw/partition_gui.py:567 ../textw/network_text.py:52
#: ../loader2/cdinstall.c:113 ../loader2/cdinstall.c:121
#: ../loader2/driverdisk.c:480
msgid "Continue"
msgstr "Continuar"

#: ../rescue.py:287 ../rescue.py:292
msgid "Read-Only"
msgstr "Só Lectura"

#: ../rescue.py:318
msgid "System to Rescue"
msgstr ""

#: ../rescue.py:319
msgid "What partition holds the root partition of your installation?"
msgstr "Que partición é a raíz da instalación?"

#: ../rescue.py:321 ../rescue.py:325 ../text.py:569 ../text.py:571
msgid "Exit"
msgstr "Saír"

#: ../rescue.py:344
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: ../rescue.py:352
#, python-format
msgid ""
"Your system has been mounted under %s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""

#: ../rescue.py:435
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""

#: ../rescue.py:441
msgid "Rescue Mode"
msgstr ""

#: ../rescue.py:442
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""

#: ../rescue.py:455
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: ../text.py:150 ../text.py:165
msgid "Save"
msgstr "Gardar"

#: ../text.py:153 ../text.py:167
msgid "Remote"
msgstr ""

#: ../text.py:155 ../text.py:163
msgid "Debug"
msgstr ""

#: ../text.py:159
msgid "Exception Occurred"
msgstr ""

#: ../text.py:188
msgid "Save to Remote Host"
msgstr ""

#: ../text.py:191
msgid "Host"
msgstr "Máquina"

#: ../text.py:193
msgid "Remote path"
msgstr ""

#: ../text.py:195
msgid "User name"
msgstr "Nome de usuario"

#: ../text.py:197
msgid "Password"
msgstr "Contrasinal"

#: ../text.py:254
msgid "Help not available"
msgstr "Axuda non dispoñible."

#: ../text.py:255
msgid "No help is available for this step of the install."
msgstr "Non hai axuda dispoñible para este paso da instalación."

#: ../text.py:362
#, python-format
msgid "Please enter your %(instkey)s"
msgstr ""

#: ../text.py:379 tmp/instkey.glade.h:4
#, no-c-format, python-format
msgid "Skip entering %(instkey)s"
msgstr ""

#: ../text.py:411
msgid "Save Crash Dump"
msgstr ""

#: ../text.py:466 ../loader2/lang.c:55 ../loader2/loader.c:155
#, c-format, python-format
msgid "Welcome to %s"
msgstr "Benvido a %s"

#: ../text.py:469
msgid ""
" <F1> for help | <Tab> between elements | <Space> selects | <F12> next screen"
msgstr ""

#: ../text.py:471
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""

#: ../upgrade.py:52
msgid "Proceed with upgrade?"
msgstr ""

#: ../upgrade.py:53
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""

#: ../upgrade.py:57
msgid "Would you like to continue with the upgrade?"
msgstr ""

#: ../upgrade.py:93
msgid "Searching"
msgstr ""

#: ../upgrade.py:94
#, python-format
msgid "Searching for %s installations..."
msgstr ""

#: ../upgrade.py:149 ../upgrade.py:157
msgid "Dirty File Systems"
msgstr ""

#: ../upgrade.py:150
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""

#: ../upgrade.py:158
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: ../upgrade.py:295 ../upgrade.py:301
msgid "Mount failed"
msgstr ""

#: ../upgrade.py:296
msgid ""
"One or more of the file systems listed in the /etc/fstab on your Linux "
"system cannot be mounted. Please fix this problem and try to upgrade again."
msgstr ""

#: ../upgrade.py:302
msgid ""
"One or more of the file systems listed in the /etc/fstab of your Linux "
"system are inconsistent and cannot be mounted.  Please fix this problem and "
"try to upgrade again."
msgstr ""

#: ../upgrade.py:319
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""

#: ../upgrade.py:325
msgid "Absolute Symlinks"
msgstr ""

#: ../upgrade.py:336
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""

#: ../upgrade.py:342
msgid "Invalid Directories"
msgstr ""

#: ../upgrade.py:349
#, python-format
msgid "%s not found"
msgstr "non se atopou %s"

#: ../urlinstall.py:38
msgid "Connecting..."
msgstr "Conectando..."

#: ../urlinstall.py:77
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file or perhaps a "
"corrupt package.  Please verify your mirror contains all required packages, "
"and try using a different one.\n"
"\n"
"If you reboot, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: ../vnc.py:47
msgid "Unable to Start X"
msgstr "Non foi posible iniciar as X"

#: ../vnc.py:48
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""

#: ../vnc.py:55 ../vnc.py:58
msgid "Use text mode"
msgstr ""

#: ../vnc.py:56
msgid "Start VNC"
msgstr "Iniciar VNC"

#: ../vnc.py:66
msgid "VNC Configuration"
msgstr "Configuración de VNC"

#: ../vnc.py:70
msgid "No password"
msgstr "Sen contrasinal"

#: ../vnc.py:73
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""

#: ../vnc.py:81 ../textw/userauth_text.py:42 ../loader2/urls.c:460
msgid "Password:"
msgstr "Contrasinal:"

#: ../vnc.py:82 ../textw/userauth_text.py:43
msgid "Password (confirm):"
msgstr "Contrasinal (confirmar):"

#: ../vnc.py:104 ../textw/userauth_text.py:64
msgid "Password Mismatch"
msgstr ""

#: ../vnc.py:105 ../textw/userauth_text.py:65
msgid "The passwords you entered were different. Please try again."
msgstr ""

#: ../vnc.py:110 ../textw/userauth_text.py:59
msgid "Password Length"
msgstr "Lonxitude do Contrasinal"

#: ../vnc.py:111
msgid "The password must be at least six characters long."
msgstr "O contrasinal debe ter como mínimo seis caracteres de lonxitude."

#: ../vnc.py:141
msgid "VNC Password Error"
msgstr ""

#: ../vnc.py:142
msgid ""
"You need to specify a vnc password of at least 6 characters long.\n"
"\n"
"Press <return> to reboot your system.\n"
msgstr ""

#: ../vnc.py:187
msgid "Starting VNC..."
msgstr "Iniciando VNC..."

#: ../vnc.py:230
#, python-format
msgid "%s %s installation on host %s"
msgstr ""

#: ../vnc.py:232
#, python-format
msgid "%s %s installation"
msgstr ""

#: ../vnc.py:252
msgid "Unable to set vnc password - using no password!"
msgstr ""

#: ../vnc.py:253
msgid "Make sure your password is at least 6 characters in length."
msgstr "Asegúrese de que o contrasinal ten alomenos 6 caracteres de lonxitude."

#: ../vnc.py:276
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: ../vnc.py:280
msgid "The VNC server is now running."
msgstr ""

#: ../vnc.py:283
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr ""

#: ../vnc.py:295
msgid "Connected!"
msgstr "Conectado!"

#: ../vnc.py:300
msgid "Giving up attempting to connect after 50 tries!\n"
msgstr ""

#: ../vnc.py:302
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""

#: ../vnc.py:304
msgid "Please manually connect your vnc client to begin the install."
msgstr ""

#: ../vnc.py:308
msgid "Will try to connect again in 15 seconds..."
msgstr ""

#: ../vnc.py:316
#, python-format
msgid "Please connect to %s to begin the install..."
msgstr ""

#: ../vnc.py:318
msgid "Please connect to begin the install..."
msgstr ""

#: ../yuminstall.py:73
#, python-format
msgid "%s MB"
msgstr ""

#: ../yuminstall.py:76
#, python-format
msgid "%s KB"
msgstr ""

#: ../yuminstall.py:79
#, python-format
msgid "%s Byte"
msgstr ""

#: ../yuminstall.py:81
#, python-format
msgid "%s Bytes"
msgstr ""

#: ../yuminstall.py:129
msgid "Processing"
msgstr "Procesando"

#: ../yuminstall.py:130
msgid "Preparing transaction from installation source..."
msgstr ""

#: ../yuminstall.py:160
#, fuzzy, python-format
msgid "<b>Installing %s</b> (%s)\n"
msgstr "_Instalar %s"

#: ../yuminstall.py:194
#, python-format
msgid "%s of %s packages completed"
msgstr ""

#: ../yuminstall.py:531 ../yuminstall.py:532
msgid "file conflicts"
msgstr ""

#: ../yuminstall.py:533
msgid "older package(s)"
msgstr ""

#: ../yuminstall.py:534
msgid "insufficient disk space"
msgstr ""

#: ../yuminstall.py:535
msgid "insufficient disk inodes"
msgstr ""

#: ../yuminstall.py:536
msgid "package conflicts"
msgstr ""

#: ../yuminstall.py:537
msgid "package already installed"
msgstr "paquete xa instalado"

#: ../yuminstall.py:538
msgid "required package"
msgstr "paquete requirido"

#: ../yuminstall.py:539
msgid "package for incorrect arch"
msgstr ""

#: ../yuminstall.py:540
msgid "package for incorrect os"
msgstr ""

#: ../yuminstall.py:554
msgid "You need more space on the following file systems:\n"
msgstr "Precisa máis espacio nos seguintes sistemas de ficheiros:\n"

#: ../yuminstall.py:570
msgid "Error running transaction"
msgstr ""

#: ../yuminstall.py:571
#, fuzzy, python-format
msgid ""
"There was an error running your transaction, for the following reason(s): %s"
msgstr ""
"O nome de máquina \"%s\" non é válido pola seguinte razón:\n"
"\n"
"%s"

#: ../yuminstall.py:575 ../yuminstall.py:608 ../yuminstall.py:744
#: ../yuminstall.py:930
msgid "Re_boot"
msgstr ""

#: ../yuminstall.py:699
msgid "Retrieving installation information..."
msgstr ""

#: ../yuminstall.py:701
#, python-format
msgid "Retrieving installation information for %s..."
msgstr ""

#: ../yuminstall.py:723
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.  %s"
msgstr ""

#: ../yuminstall.py:739
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""

#: ../yuminstall.py:771
msgid "Uncategorized"
msgstr ""

#: ../yuminstall.py:924
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or reboot."
msgstr ""

#: ../yuminstall.py:1085
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""

#: ../yuminstall.py:1113
msgid "Install Starting"
msgstr ""

#: ../yuminstall.py:1114
msgid "Starting install process.  This may take several minutes..."
msgstr "Iniciando o proceso de instalación.  Isto pode levar varios minutos..."

#: ../yuminstall.py:1130
msgid "Post Upgrade"
msgstr ""

#: ../yuminstall.py:1131
msgid "Performing post upgrade configuration..."
msgstr ""

#: ../yuminstall.py:1133
msgid "Post Install"
msgstr ""

#: ../yuminstall.py:1134
msgid "Performing post install configuration..."
msgstr ""

#: ../yuminstall.py:1327
msgid "Installation Progress"
msgstr "Progreso da Instalación"

#: ../yuminstall.py:1362
msgid "Dependency Check"
msgstr ""

#: ../yuminstall.py:1363
msgid "Checking dependencies in packages selected for installation..."
msgstr ""

#: ../zfcp.py:43
msgid "You have not specified a device number or the number is invalid"
msgstr ""

#: ../zfcp.py:45
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr ""

#: ../zfcp.py:47
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr ""

#: ../iw/account_gui.py:26
msgid "<b>Caps Lock is on.</b>"
msgstr ""

#: ../iw/account_gui.py:33
msgid "Set Root Password"
msgstr "Establecer o Contrasinal de Root"

#: ../iw/account_gui.py:48 ../iw/account_gui.py:56 ../iw/account_gui.py:63
#: ../iw/account_gui.py:72 ../textw/userauth_text.py:69
msgid "Error with Password"
msgstr "Erro co Contrasinal"

#: ../iw/account_gui.py:49
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""
"Debe introducir o contrasinal de root e confirmalo escribíndoo unha segunda "
"vez para continuar."

#: ../iw/account_gui.py:57
msgid "The passwords you entered were different.  Please try again."
msgstr "Os contrasinais que introduciu son diferentes.  Ténteo de novo."

#: ../iw/account_gui.py:64
msgid "The root password must be at least six characters long."
msgstr ""
"O contrasinal de root debe ter polo menos seis caracteres de lonxitude."

#: ../iw/account_gui.py:73 ../textw/userauth_text.py:70
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""

#: ../iw/account_gui.py:107
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: ../iw/account_gui.py:122
msgid "Root _Password: "
msgstr ""

#: ../iw/account_gui.py:125
msgid "_Confirm: "
msgstr "_Confirmar: "

#: ../iw/autopart_type.py:147
msgid "Invalid Initiator Name"
msgstr ""

#: ../iw/autopart_type.py:148
msgid "You must provide an initiator name."
msgstr ""

#: ../iw/autopart_type.py:170
msgid "Error with Data"
msgstr ""

#: ../iw/autopart_type.py:259 ../textw/partition_text.py:1541
#, fuzzy
msgid "Remove all partitions on selected drives and create default layout"
msgstr "Eliminar tódalas particións deste sistema"

#: ../iw/autopart_type.py:260 ../textw/partition_text.py:1542
#, fuzzy
msgid "Remove Linux partitions on selected drives and create default layout"
msgstr "Eliminar tódalas particións Linux deste sistema"

#: ../iw/autopart_type.py:261 ../textw/partition_text.py:1543
msgid "Use free space on selected drives and create default layout"
msgstr ""

#: ../iw/autopart_type.py:262 ../textw/partition_text.py:1544
msgid "Create custom layout"
msgstr ""

#: ../iw/blpasswidget.py:37
msgid ""
"A boot loader password prevents users from changing options passed to the "
"kernel.  For greater system security, it is recommended that you set a "
"password."
msgstr ""

#: ../iw/blpasswidget.py:42
msgid "_Use a boot loader password"
msgstr ""

#: ../iw/blpasswidget.py:73
msgid "Change _password"
msgstr ""

#: ../iw/blpasswidget.py:96
msgid "Enter Boot Loader Password"
msgstr ""

#: ../iw/blpasswidget.py:102
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""

#: ../iw/blpasswidget.py:109
msgid "_Password:"
msgstr ""

#: ../iw/blpasswidget.py:115
msgid "Con_firm:"
msgstr "Con_firmar:"

#: ../iw/blpasswidget.py:136
#, fuzzy
msgid "Passwords don't match"
msgstr "Os contrasinais non coinciden"

#: ../iw/blpasswidget.py:137 ../textw/bootloader_text.py:439
msgid "Passwords do not match"
msgstr ""

#: ../iw/blpasswidget.py:146 ../textw/bootloader_text.py:449
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: ../iw/bootloader_advanced_gui.py:27
msgid "Advanced Boot Loader Configuration"
msgstr ""

#: ../iw/bootloader_advanced_gui.py:42 ../textw/bootloader_text.py:124
msgid ""
"If LBA32 is not supported by your system's BIOS, forcing its use can prevent "
"your machine from booting.\n"
"\n"
"Would you like to continue and force LBA32 mode?"
msgstr ""

#: ../iw/bootloader_advanced_gui.py:48
msgid "Force LBA32"
msgstr "Forzar LBA32"

#: ../iw/bootloader_advanced_gui.py:69
msgid "_Force LBA32 (not normally required)"
msgstr ""

#: ../iw/bootloader_advanced_gui.py:73
msgid ""
"If you wish to add default options to the boot command, enter them into the "
"'General kernel parameters' field."
msgstr ""

#: ../iw/bootloader_advanced_gui.py:79
msgid "_General kernel parameters"
msgstr ""

#: ../iw/bootloader_main_gui.py:30 ../textw/bootloader_text.py:42
#: ../textw/bootloader_text.py:107 ../textw/bootloader_text.py:164
#: ../textw/bootloader_text.py:282 ../textw/bootloader_text.py:391
msgid "Boot Loader Configuration"
msgstr "Configuración do Cargador de Arrinque"

#: ../iw/bootloader_main_gui.py:117
#, python-format
msgid "The %s boot loader will be installed on /dev/%s."
msgstr ""

#: ../iw/bootloader_main_gui.py:123
msgid "No boot loader will be installed."
msgstr ""

#: ../iw/bootloader_main_gui.py:152
msgid "Configure advanced boot loader _options"
msgstr ""

#: ../iw/bootlocwidget.py:39
msgid "Install Boot Loader record on:"
msgstr ""

#: ../iw/bootlocwidget.py:72
msgid "_Change Drive Order"
msgstr ""

#: ../iw/bootlocwidget.py:84
msgid "Edit Drive Order"
msgstr ""

#: ../iw/bootlocwidget.py:89
msgid ""
"Arrange the drives to be in the same order as used by the BIOS. Changing the "
"drive order may be useful if you have multiple SCSI adapters or both SCSI "
"and IDE adapters and want to boot from the SCSI device.\n"
"\n"
"Changing the drive order will change where the installation program locates "
"the Master Boot Record (MBR)."
msgstr ""

#: ../iw/confirm_gui.py:29 ../textw/confirm_text.py:35
#: ../textw/confirm_text.py:63
msgid "Reboot?"
msgstr "Reiniciar?"

#: ../iw/confirm_gui.py:30 ../textw/confirm_text.py:36
#: ../textw/confirm_text.py:64
msgid "The system will be rebooted now."
msgstr ""

#: ../iw/confirm_gui.py:74
msgid "About to Install"
msgstr ""

#: ../iw/confirm_gui.py:80
#, python-format
msgid "Click next to begin installation of %s."
msgstr "Prema seguinte para comezar a instalación de %s."

#: ../iw/confirm_gui.py:81
#, python-format
msgid ""
"A complete log of the installation can be found in the file '%s' after "
"rebooting your system.\n"
"\n"
"A kickstart file containing the installation options selected can be found "
"in the file '%s' after rebooting the system."
msgstr ""

#: ../iw/confirm_gui.py:88
msgid "About to Upgrade"
msgstr ""

#: ../iw/confirm_gui.py:94
#, python-format
msgid "Click next to begin upgrade of %s."
msgstr "Prema seguinte para comezar a actualización de %s."

#: ../iw/confirm_gui.py:95
#, python-format
msgid ""
"A complete log of the upgrade can be found in the file '%s' after rebooting "
"your system."
msgstr ""

#: ../iw/congrats_gui.py:24
msgid "Congratulations"
msgstr "Noraboa"

#: ../iw/congrats_gui.py:60
#, fuzzy
msgid ""
"Press the \"Reboot\" button to reboot your system.\n"
"\n"
msgstr ""
"\n"
"\n"
"Prema 'Aceptar' para reiniciar o sistema."

#: ../iw/congrats_gui.py:63
#, python-format
msgid ""
"Congratulations, the installation is complete.\n"
"\n"
"%s%s"
msgstr ""
"Noraboa, a instalación está completada.\n"
"\n"
"%s%s"

#: ../iw/driveorderwidget.py:44
msgid "Drive"
msgstr "Unidade"

#: ../iw/driveorderwidget.py:44 ../textw/partition_text.py:1447
msgid "Size"
msgstr "Tamaño"

#: ../iw/driveorderwidget.py:44
msgid "Model"
msgstr "Modelo"

#: ../iw/examine_gui.py:28
msgid "Upgrade Examine"
msgstr ""

#: ../iw/examine_gui.py:50
#, python-format
msgid "_Install %s"
msgstr "_Instalar %s"

#: ../iw/examine_gui.py:52
msgid ""
"Choose this option to freshly install your system. Existing software and "
"data may be overwritten depending on your configuration choices."
msgstr ""

#: ../iw/examine_gui.py:56
msgid "_Upgrade an existing installation"
msgstr ""

#: ../iw/examine_gui.py:58
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option preserves the existing data on your drives."
msgstr ""

#: ../iw/examine_gui.py:105 ../iw/pixmapRadioButtonGroup_gui.py:197
msgid "The following installed system will be upgraded:"
msgstr ""

#: ../iw/examine_gui.py:118
msgid "Unknown Linux system"
msgstr "Sistema Linux descoñecido"

#: ../iw/language_gui.py:24 ../textw/language_text.py:41
msgid "Language Selection"
msgstr "Selección de Lingua"

#: ../iw/language_gui.py:67 ../textw/language_text.py:42 ../loader2/lang.c:367
msgid "What language would you like to use during the installation process?"
msgstr ""

#: ../iw/lvm_dialog_gui.py:115 ../iw/lvm_dialog_gui.py:162
#: ../iw/lvm_dialog_gui.py:173 ../iw/lvm_dialog_gui.py:213
#: ../iw/lvm_dialog_gui.py:287 ../iw/lvm_dialog_gui.py:601
#: ../iw/lvm_dialog_gui.py:669 ../iw/lvm_dialog_gui.py:880
#: ../textw/partition_text.py:1301 ../textw/partition_text.py:1320
msgid "Not enough space"
msgstr "Non hai espacio suficiente"

#: ../iw/lvm_dialog_gui.py:116
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""

#: ../iw/lvm_dialog_gui.py:125
msgid "Confirm Physical Extent Change"
msgstr ""

#: ../iw/lvm_dialog_gui.py:126
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""

#: ../iw/lvm_dialog_gui.py:135 ../iw/lvm_dialog_gui.py:195
#: ../iw/network_gui.py:143 ../iw/network_gui.py:147 ../iw/network_gui.py:167
msgid "C_ontinue"
msgstr "C_ontinuar"

#: ../iw/lvm_dialog_gui.py:163
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is larger than the smallest physical volume (%10.2f MB) in the "
"volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:174
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected (%"
"10.2f MB) is too large compared to the size of the smallest physical volume "
"(%10.2f MB) in the volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:188
msgid "Too small"
msgstr ""

#: ../iw/lvm_dialog_gui.py:189
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:214
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""

#: ../iw/lvm_dialog_gui.py:288
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""

#: ../iw/lvm_dialog_gui.py:368 ../textw/partition_text.py:1139
msgid "Make Logical Volume"
msgstr ""

#: ../iw/lvm_dialog_gui.py:371
#, python-format
msgid "Edit Logical Volume: %s"
msgstr ""

#: ../iw/lvm_dialog_gui.py:373 ../textw/partition_text.py:1137
msgid "Edit Logical Volume"
msgstr ""

#: ../iw/lvm_dialog_gui.py:386 ../iw/partition_dialog_gui.py:293
#: ../iw/raid_dialog_gui.py:280
msgid "_Mount Point:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:394 ../iw/raid_dialog_gui.py:289
msgid "_File System Type:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:404 ../iw/partition_dialog_gui.py:312
#: ../iw/raid_dialog_gui.py:299
msgid "Original File System Type:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:409 ../iw/partition_dialog_gui.py:323
#: ../iw/raid_dialog_gui.py:304
msgid "Unknown"
msgstr "Descoñecido"

#: ../iw/lvm_dialog_gui.py:415 ../iw/partition_dialog_gui.py:358
#: ../iw/raid_dialog_gui.py:310
msgid "Original File System Label:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:424
msgid "_Logical Volume Name:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:432 ../textw/partition_text.py:287
msgid "Logical Volume Name:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:440 ../iw/partition_dialog_gui.py:370
msgid "_Size (MB):"
msgstr ""

#: ../iw/lvm_dialog_gui.py:446 ../iw/partition_dialog_gui.py:387
#: ../iw/partition_dialog_gui.py:430 ../textw/partition_text.py:302
#: ../textw/partition_text.py:379 ../textw/partition_text.py:462
#: ../textw/partition_text.py:570
msgid "Size (MB):"
msgstr "Tamaño (MB):"

#: ../iw/lvm_dialog_gui.py:461
#, python-format
msgid "(Max size is %s MB)"
msgstr ""

#: ../iw/lvm_dialog_gui.py:522 ../textw/partition_text.py:1290
msgid "Illegal size"
msgstr ""

#: ../iw/lvm_dialog_gui.py:523 ../textw/partition_text.py:1291
msgid "The requested size as entered is not a valid number greater than 0."
msgstr ""

#: ../iw/lvm_dialog_gui.py:556
msgid "Mount point in use"
msgstr ""

#: ../iw/lvm_dialog_gui.py:557
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""

#: ../iw/lvm_dialog_gui.py:568 ../textw/partition_text.py:1261
msgid "Illegal Logical Volume Name"
msgstr ""

#: ../iw/lvm_dialog_gui.py:587 ../textw/partition_text.py:1278
msgid "Illegal logical volume name"
msgstr ""

#: ../iw/lvm_dialog_gui.py:588 ../textw/partition_text.py:1279
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""

#: ../iw/lvm_dialog_gui.py:602
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). To increase this limit you can create more Physical "
"Volumes from unpartitioned disk space and add them to this Volume Group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:646 ../iw/partition_dialog_gui.py:179
#: ../iw/partition_dialog_gui.py:191 ../iw/partition_dialog_gui.py:239
#: ../iw/raid_dialog_gui.py:207 ../textw/partition_text.py:911
#: ../textw/partition_text.py:933 ../textw/partition_text.py:1107
#: ../textw/partition_text.py:1340
msgid "Error With Request"
msgstr ""

#: ../iw/lvm_dialog_gui.py:670 ../iw/lvm_dialog_gui.py:881
#, python-format
msgid ""
"The logical volumes you have configured require %d MB, but the volume group "
"only has %d MB.  Please either make the volume group larger or make the "
"logical volume(s) smaller."
msgstr ""

#: ../iw/lvm_dialog_gui.py:720
msgid "No free slots"
msgstr ""

#: ../iw/lvm_dialog_gui.py:721
#, python-format
msgid "You cannot create more than %s logical volumes per volume group."
msgstr ""

#: ../iw/lvm_dialog_gui.py:727
msgid "No free space"
msgstr "Non hai espacio baleiro"

#: ../iw/lvm_dialog_gui.py:728
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""

#: ../iw/lvm_dialog_gui.py:756
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr ""

#: ../iw/lvm_dialog_gui.py:892
msgid "Invalid Volume Group Name"
msgstr ""

#: ../iw/lvm_dialog_gui.py:903
msgid "Name in use"
msgstr ""

#: ../iw/lvm_dialog_gui.py:904
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""

#: ../iw/lvm_dialog_gui.py:947
msgid "Not enough physical volumes"
msgstr ""

#: ../iw/lvm_dialog_gui.py:948
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""

#: ../iw/lvm_dialog_gui.py:959
msgid "Make LVM Volume Group"
msgstr ""

#: ../iw/lvm_dialog_gui.py:962
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr ""

#: ../iw/lvm_dialog_gui.py:964
msgid "Edit LVM Volume Group"
msgstr ""

#: ../iw/lvm_dialog_gui.py:980
msgid "_Volume Group Name:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:988
msgid "Volume Group Name:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:996
msgid "_Physical Extent:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:1011
msgid "Physical Volumes to _Use:"
msgstr ""

#: ../iw/lvm_dialog_gui.py:1017
msgid "Used Space:"
msgstr "Espacio Usado:"

#: ../iw/lvm_dialog_gui.py:1034
msgid "Free Space:"
msgstr "Espacio Baleiro:"

#: ../iw/lvm_dialog_gui.py:1052
msgid "Total Space:"
msgstr "Espacio Total:"

#: ../iw/lvm_dialog_gui.py:1081
msgid "Logical Volume Name"
msgstr ""

#: ../iw/lvm_dialog_gui.py:1084 ../iw/partition_gui.py:364
#: ../iw/upgrade_swap_gui.py:136 ../textw/partition_text.py:1447
#: ../textw/upgrade_text.py:106
msgid "Mount Point"
msgstr "Punto de Montaxe"

#: ../iw/lvm_dialog_gui.py:1087 ../iw/partition_gui.py:369
msgid "Size (MB)"
msgstr "Tamaño (MB)"

#: ../iw/lvm_dialog_gui.py:1101 ../iw/osbootwidget.py:96
msgid "_Add"
msgstr "_Engadir"

#: ../iw/lvm_dialog_gui.py:1104 ../iw/network_gui.py:448
#: ../iw/osbootwidget.py:100 ../iw/partition_gui.py:1352
msgid "_Edit"
msgstr "_Editar"

#: ../iw/lvm_dialog_gui.py:1119
msgid "Logical Volumes"
msgstr ""

#: ../iw/mouse_gui.py:24
msgid "Mouse Configuration"
msgstr "Configuración do Rato"

#: ../iw/mouse_gui.py:77 ../textw/mouse_text.py:20
msgid "/dev/ttyS0 (COM1 under DOS)"
msgstr "/dev/ttyS0 (COM1 en DOS)"

#: ../iw/mouse_gui.py:78 ../textw/mouse_text.py:21
msgid "/dev/ttyS1 (COM2 under DOS)"
msgstr "/dev/ttyS1 (COM2 en DOS)"

#: ../iw/mouse_gui.py:79 ../textw/mouse_text.py:22
msgid "/dev/ttyS2 (COM3 under DOS)"
msgstr "/dev/ttyS2 (COM3 en DOS)"

#: ../iw/mouse_gui.py:80 ../textw/mouse_text.py:23
msgid "/dev/ttyS3 (COM4 under DOS)"
msgstr "/dev/ttyS3 (COM4 en DOS)"

#: ../iw/mouse_gui.py:90 ../iw/osbootwidget.py:157
msgid "_Device"
msgstr "_Dispositivo"

#: ../iw/mouse_gui.py:136
msgid "_Model"
msgstr "_Modelo"

#: ../iw/mouse_gui.py:234
msgid "_Emulate 3 buttons"
msgstr ""

#: ../iw/mouse_gui.py:249
msgid "Select the appropriate mouse for the system."
msgstr ""

#: ../iw/netconfig_dialog.py:135 ../iw/network_gui.py:142
#: ../iw/network_gui.py:146 ../iw/network_gui.py:150 ../iw/network_gui.py:154
#: ../iw/network_gui.py:162 ../iw/network_gui.py:167
#: ../textw/network_text.py:44 ../textw/network_text.py:49
#: ../textw/network_text.py:55
msgid "Error With Data"
msgstr ""

#: ../iw/netconfig_dialog.py:137
#, python-format
msgid ""
"An error occurred converting the value entered for \"%s\":\n"
"%s"
msgstr ""

#: ../iw/netconfig_dialog.py:155
msgid "Dynamic IP"
msgstr ""

#: ../iw/netconfig_dialog.py:156 ../loader2/net.c:409 ../loader2/net.c:830
#, c-format, python-format
msgid "Sending request for IP information for %s..."
msgstr ""

#: ../iw/netconfig_dialog.py:176 ../textw/network_text.py:85
#: tmp/netpostconfig.glade.h:12
msgid "IP Address"
msgstr ""

#: ../iw/netconfig_dialog.py:182
msgid "Netmask"
msgstr ""

#: ../iw/netconfig_dialog.py:188 ../iw/network_gui.py:27
#: ../iw/network_gui.py:524
msgid "Gateway"
msgstr "Pasarela"

#: ../iw/netconfig_dialog.py:195
msgid "Nameserver"
msgstr ""

#: ../iw/netconfig_dialog.py:214
msgid "Error configuring network device"
msgstr ""

#: ../iw/network_gui.py:27 ../iw/network_gui.py:526
msgid "Primary DNS"
msgstr ""

#: ../iw/network_gui.py:27 ../iw/network_gui.py:528
msgid "Secondary DNS"
msgstr ""

#: ../iw/network_gui.py:28
msgid "_Gateway"
msgstr "_Pasarela"

#: ../iw/network_gui.py:28
msgid "_Primary DNS"
msgstr ""

#: ../iw/network_gui.py:28
msgid "_Secondary DNS"
msgstr ""

#: ../iw/network_gui.py:31 ../textw/network_text.py:524
msgid "Network Configuration"
msgstr "Configuración da Rede"

#: ../iw/network_gui.py:143
msgid ""
"You have not specified a hostname.  Depending on your network environment "
"this may cause problems later."
msgstr ""

#: ../iw/network_gui.py:147
#, python-format
msgid ""
"You have not specified the field \"%s\".  Depending on your network "
"environment this may cause problems later."
msgstr ""

#: ../iw/network_gui.py:151 ../textw/network_text.py:808
#, python-format
msgid ""
"The hostname \"%s\" is not valid for the following reason:\n"
"\n"
"%s"
msgstr ""
"O nome de máquina \"%s\" non é válido pola seguinte razón:\n"
"\n"
"%s"

#: ../iw/network_gui.py:155 ../textw/network_text.py:45
#, python-format
msgid "A value is required for the field %s."
msgstr ""

#: ../iw/network_gui.py:158 ../textw/network_text.py:35
#, python-format
msgid "Error With %s Data"
msgstr ""

#: ../iw/network_gui.py:159 ../textw/network_text.py:36
#: ../textw/network_text.py:584 ../textw/network_text.py:588
#: ../textw/welcome_text.py:22
#, python-format
msgid "%s"
msgstr "%s"

#: ../iw/network_gui.py:163 ../textw/network_text.py:56
msgid "The IPv4 information you have entered is invalid."
msgstr ""

#: ../iw/network_gui.py:167
msgid ""
"You have no active network devices.  Your system will not be able to "
"communicate over a network by default without at least one device active."
msgstr ""

#: ../iw/network_gui.py:292 ../iw/network_gui.py:306
msgid "Disabled"
msgstr ""

#: ../iw/network_gui.py:389
msgid "Active on Boot"
msgstr "Activar ó Arrincar"

#: ../iw/network_gui.py:391 ../iw/osbootwidget.py:67
#: ../iw/partition_gui.py:363 ../textw/bootloader_text.py:192
#: ../textw/bootloader_text.py:258 ../textw/mouse_text.py:38
#: ../textw/partition_text.py:1447
msgid "Device"
msgstr "Dispositivo"

#: ../iw/network_gui.py:393
#, fuzzy
msgid "IPv4/Netmask"
msgstr "IP/Máscara de rede"

#: ../iw/network_gui.py:395
msgid "IPv6/Prefix"
msgstr ""

#: ../iw/network_gui.py:455
msgid "Network Devices"
msgstr "Dispositivos de Rede"

#: ../iw/network_gui.py:466
msgid "Set the hostname:"
msgstr ""

#: ../iw/network_gui.py:471
msgid "_automatically via DHCP"
msgstr ""

#: ../iw/network_gui.py:478
msgid "_manually"
msgstr "_manualmente"

#: ../iw/network_gui.py:483
msgid "(e.g., host.domain.com)"
msgstr ""

#: ../iw/network_gui.py:489
msgid "Hostname"
msgstr "Nome de máquina"

#: ../iw/network_gui.py:533
msgid "Miscellaneous Settings"
msgstr ""

#: ../iw/network_gui.py:621
#, fuzzy
msgid "Edit Device "
msgstr "Dispositivo"

#: ../iw/network_gui.py:625
msgid "Unknown Ethernet Device"
msgstr ""

#: ../iw/network_gui.py:631
#, fuzzy
msgid "unknown"
msgstr "Descoñecido"

#: ../iw/network_gui.py:633
msgid "Hardware address: "
msgstr ""

#: ../iw/network_gui.py:769 ../textw/network_text.py:220 ../loader2/net.c:791
msgid "Missing Protocol"
msgstr ""

#: ../iw/network_gui.py:770 ../textw/network_text.py:221
msgid "You must select at least IPv4 or IPv6 support."
msgstr ""

#: ../iw/network_gui.py:793 ../iw/network_gui.py:834 ../iw/network_gui.py:840
#: ../textw/network_text.py:66 ../loader2/net.c:93
msgid "Invalid Prefix"
msgstr ""

#: ../iw/network_gui.py:794
msgid "IPv4 prefix must be between 0 and 32."
msgstr ""

#: ../iw/network_gui.py:835 ../iw/network_gui.py:841
msgid "IPv6 prefix must be between 0 and 128."
msgstr ""

#: ../iw/osbootwidget.py:43
msgid ""
"You can configure the boot loader to boot other operating systems by "
"selecting from the list. To add an operating systems that was not "
"automatically detected, click 'Add.' To change the operating system booted "
"by default, select 'Default' next to the desired operating system."
msgstr ""

#: ../iw/osbootwidget.py:67 ../textw/bootloader_text.py:258
msgid "Default"
msgstr "Predeterminado"

#: ../iw/osbootwidget.py:67
msgid "Label"
msgstr "Etiqueta"

#: ../iw/osbootwidget.py:130
msgid "Image"
msgstr "Imaxe"

#: ../iw/osbootwidget.py:137
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""

#: ../iw/osbootwidget.py:149
msgid "_Label"
msgstr ""

#: ../iw/osbootwidget.py:188
msgid "Default Boot _Target"
msgstr ""

#: ../iw/osbootwidget.py:217
msgid "You must specify a label for the entry"
msgstr "Debe especificar unha etiqueta para a entrada"

#: ../iw/osbootwidget.py:226
msgid "Boot label contains illegal characters"
msgstr ""

#: ../iw/osbootwidget.py:250
msgid "Duplicate Label"
msgstr ""

#: ../iw/osbootwidget.py:251
msgid "This label is already in use for another boot entry."
msgstr ""

#: ../iw/osbootwidget.py:264
msgid "Duplicate Device"
msgstr ""

#: ../iw/osbootwidget.py:265
msgid "This device is already being used for another boot entry."
msgstr ""

#: ../iw/osbootwidget.py:329 ../textw/bootloader_text.py:341
msgid "Cannot Delete"
msgstr "Non se puido Eliminar"

#: ../iw/osbootwidget.py:330 ../textw/bootloader_text.py:342
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""

#: ../iw/partition_dialog_gui.py:58
msgid "Additional Size Options"
msgstr ""

#: ../iw/partition_dialog_gui.py:63
msgid "_Fixed size"
msgstr ""

#: ../iw/partition_dialog_gui.py:65
msgid "Fill all space _up to (MB):"
msgstr ""

#: ../iw/partition_dialog_gui.py:75
msgid "Fill to maximum _allowable size"
msgstr ""

#: ../iw/partition_dialog_gui.py:180
msgid "The end cylinder must be greater than the start cylinder."
msgstr ""

#: ../iw/partition_dialog_gui.py:270 ../textw/partition_text.py:712
msgid "Add Partition"
msgstr ""

#: ../iw/partition_dialog_gui.py:273
#, python-format
msgid "Edit Partition: /dev/%s"
msgstr ""

#: ../iw/partition_dialog_gui.py:275
msgid "Edit Partition"
msgstr ""

#: ../iw/partition_dialog_gui.py:302
msgid "File System _Type:"
msgstr ""

#: ../iw/partition_dialog_gui.py:334
msgid "Allowable _Drives:"
msgstr ""

#: ../iw/partition_dialog_gui.py:349
msgid "Drive:"
msgstr ""

#: ../iw/partition_dialog_gui.py:393
msgid "_Start Cylinder:"
msgstr ""

#: ../iw/partition_dialog_gui.py:411
msgid "_End Cylinder:"
msgstr ""

#: ../iw/partition_dialog_gui.py:463
msgid "Force to be a _primary partition"
msgstr ""

#: ../iw/partition_gui.py:314
#, python-format
msgid "Drive %s (Geom: %s/%s/%s) (Model: %s)"
msgstr ""

#: ../iw/partition_gui.py:321
#, python-format
msgid "Drive %s (%-0.f MB) (Model: %s)"
msgstr ""

#: ../iw/partition_gui.py:365 ../textw/partition_text.py:1447
msgid "Type"
msgstr ""

#: ../iw/partition_gui.py:370 ../textw/partition_text.py:1447
msgid "Start"
msgstr ""

#: ../iw/partition_gui.py:371 ../textw/partition_text.py:1447
msgid "End"
msgstr ""

#: ../iw/partition_gui.py:408
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""

#: ../iw/partition_gui.py:410
msgid ""
"Size\n"
"(MB)"
msgstr ""

#: ../iw/partition_gui.py:542 ../textw/partition_text.py:1441
msgid "Partitioning"
msgstr ""

#: ../iw/partition_gui.py:633
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""

#: ../iw/partition_gui.py:635
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""

#: ../iw/partition_gui.py:641
msgid "Partitioning Errors"
msgstr ""

#: ../iw/partition_gui.py:647
msgid "The partitioning scheme you requested generated the following warnings."
msgstr ""

#: ../iw/partition_gui.py:649
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: ../iw/partition_gui.py:654
msgid "Partitioning Warnings"
msgstr ""

#: ../iw/partition_gui.py:676
msgid "Format Warnings"
msgstr ""

#: ../iw/partition_gui.py:681
msgid "_Format"
msgstr ""

#: ../iw/partition_gui.py:718
msgid "LVM Volume Groups"
msgstr ""

#: ../iw/partition_gui.py:753
msgid "RAID Devices"
msgstr ""

#: ../iw/partition_gui.py:781 ../iw/partition_gui.py:907
#: ../textw/partition_text.py:97 ../textw/partition_text.py:160
msgid "None"
msgstr ""

#: ../iw/partition_gui.py:799 ../loader2/hdinstall.c:265
msgid "Hard Drives"
msgstr ""

#: ../iw/partition_gui.py:870 ../textw/partition_text.py:142
#: ../textw/partition_text.py:181
msgid "Free space"
msgstr ""

#: ../iw/partition_gui.py:872 ../textw/partition_text.py:144
msgid "Extended"
msgstr ""

#: ../iw/partition_gui.py:874 ../textw/partition_text.py:146
msgid "software RAID"
msgstr ""

#: ../iw/partition_gui.py:909
msgid "Free"
msgstr ""

#: ../iw/partition_gui.py:998 ../textw/partition_text.py:229
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr ""

#: ../iw/partition_gui.py:1007
#, python-format
msgid "Warning: %s."
msgstr ""

#: ../iw/partition_gui.py:1189 ../iw/partition_gui.py:1203
msgid "Not supported"
msgstr ""

#: ../iw/partition_gui.py:1190
msgid "LVM is NOT supported on this platform."
msgstr ""

#: ../iw/partition_gui.py:1204
msgid "Software RAID is NOT supported on this platform."
msgstr ""

#: ../iw/partition_gui.py:1211
msgid "No RAID minor device numbers available"
msgstr ""

#: ../iw/partition_gui.py:1212
msgid ""
"A software RAID device cannot be created because all of the available RAID "
"minor device numbers have been used."
msgstr ""

#: ../iw/partition_gui.py:1226
msgid "RAID Options"
msgstr ""

#: ../iw/partition_gui.py:1237
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
"\n"
"You currently have %s software RAID partition(s) free to use.\n"
"\n"
msgstr ""

#: ../iw/partition_gui.py:1248
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""

#: ../iw/partition_gui.py:1254
msgid "What do you want to do now?"
msgstr ""

#: ../iw/partition_gui.py:1263
msgid "Create a software RAID _partition."
msgstr ""

#: ../iw/partition_gui.py:1266
#, python-format
msgid "Create a RAID _device [default=/dev/md%s]."
msgstr ""

#: ../iw/partition_gui.py:1270
#, python-format
msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
msgstr ""

#: ../iw/partition_gui.py:1309
msgid "Couldn't Create Drive Clone Editor"
msgstr ""

#: ../iw/partition_gui.py:1310
msgid "The drive clone editor could not be created for some reason."
msgstr ""

#: ../iw/partition_gui.py:1351
msgid "Ne_w"
msgstr ""

#: ../iw/partition_gui.py:1354
msgid "Re_set"
msgstr ""

#: ../iw/partition_gui.py:1355
msgid "R_AID"
msgstr ""

#: ../iw/partition_gui.py:1356
msgid "_LVM"
msgstr ""

#: ../iw/partition_gui.py:1397
msgid "Hide RAID device/LVM Volume _Group members"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:91 ../iw/partition_ui_helpers_gui.py:112
#: ../iw/partition_ui_helpers_gui.py:114 ../textw/partition_text.py:252
#: ../textw/partition_text.py:254 ../textw/partition_text.py:256
#: ../textw/partition_text.py:281
msgid "<Not Applicable>"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:255
msgid "How would you like to prepare the file system on this partition?"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:263
msgid "Leave _unchanged (preserve data)"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:269
msgid "_Format partition as:"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:292
msgid "Mi_grate partition to:"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:314
msgid "Check for _bad blocks?"
msgstr ""

#: ../iw/partition_ui_helpers_gui.py:349
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""

#: ../iw/partmethod_gui.py:25 ../textw/partmethod_text.py:24
msgid "Disk Partitioning Setup"
msgstr ""

#: ../iw/partmethod_gui.py:50
msgid "_Automatically partition"
msgstr ""

#: ../iw/partmethod_gui.py:53
msgid "Manually partition with _Disk Druid"
msgstr ""

#: ../iw/progress_gui.py:31
msgid "Installing Packages"
msgstr ""

#: ../iw/raid_dialog_gui.py:247
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""

#: ../iw/raid_dialog_gui.py:261 ../iw/raid_dialog_gui.py:682
#: ../textw/partition_text.py:968
msgid "Make RAID Device"
msgstr ""

#: ../iw/raid_dialog_gui.py:264
#, python-format
msgid "Edit RAID Device: /dev/md%s"
msgstr ""

#: ../iw/raid_dialog_gui.py:266 ../textw/partition_text.py:966
msgid "Edit RAID Device"
msgstr ""

#: ../iw/raid_dialog_gui.py:318
msgid "RAID _Device:"
msgstr ""

#: ../iw/raid_dialog_gui.py:336
msgid "RAID _Level:"
msgstr ""

#: ../iw/raid_dialog_gui.py:377
msgid "_RAID Members:"
msgstr ""

#: ../iw/raid_dialog_gui.py:394
msgid "Number of _spares:"
msgstr ""

#: ../iw/raid_dialog_gui.py:404
msgid "_Format partition?"
msgstr ""

#: ../iw/raid_dialog_gui.py:483
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""

#: ../iw/raid_dialog_gui.py:487 ../iw/raid_dialog_gui.py:493
#: ../iw/raid_dialog_gui.py:505 ../iw/raid_dialog_gui.py:518
msgid "Source Drive Error"
msgstr ""

#: ../iw/raid_dialog_gui.py:494
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""

#: ../iw/raid_dialog_gui.py:506
#, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive /dev/%s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""

#: ../iw/raid_dialog_gui.py:519
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""

#: ../iw/raid_dialog_gui.py:532 ../iw/raid_dialog_gui.py:538
#: ../iw/raid_dialog_gui.py:557
msgid "Target Drive Error"
msgstr ""

#: ../iw/raid_dialog_gui.py:533
msgid "Please select the target drives for the clone operation."
msgstr ""

#: ../iw/raid_dialog_gui.py:539
#, python-format
msgid "The source drive /dev/%s cannot be selected as a target drive as well."
msgstr ""

#: ../iw/raid_dialog_gui.py:558
#, python-format
msgid ""
"The target drive /dev/%s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""

#: ../iw/raid_dialog_gui.py:619
msgid "Please select a source drive."
msgstr ""

#: ../iw/raid_dialog_gui.py:639
#, python-format
msgid ""
"The drive /dev/%s will now be cloned to the following drives:\n"
"\n"
msgstr ""

#: ../iw/raid_dialog_gui.py:644
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""

#: ../iw/raid_dialog_gui.py:647
msgid "Final Warning"
msgstr ""

#: ../iw/raid_dialog_gui.py:649
msgid "Clone Drives"
msgstr ""

#: ../iw/raid_dialog_gui.py:658
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr ""

#: ../iw/raid_dialog_gui.py:692
msgid ""
"Clone Drive Tool\n"
"\n"
"This tool allows you to significantly reduce the amount of effort required "
"to setup RAID arrays.  This tool uses a source drive which has been prepared "
"with the desired partitioning layout, and clones this layout onto other "
"similar sized drives.  Then a RAID device can be created.\n"
"\n"
"NOTE: The source drive must have partitions which are restricted to be on "
"that drive only, and can only contain unused software RAID partitions.  "
"Other partition types are not allowed.\n"
"\n"
"EVERYTHING on the target drive(s) will be destroyed by this process."
msgstr ""

#: ../iw/raid_dialog_gui.py:712
msgid "Source Drive:"
msgstr ""

#: ../iw/raid_dialog_gui.py:720
msgid "Target Drive(s):"
msgstr ""

#: ../iw/raid_dialog_gui.py:728
msgid "Drives"
msgstr ""

#: ../iw/release_notes.py:151 ../iw/release_notes.py:156
msgid "Release notes are missing.\n"
msgstr ""

#: ../iw/release_notes.py:236
msgid "Release Notes"
msgstr ""

#: ../iw/release_notes.py:239
msgid "Unable to load file!"
msgstr ""

#: ../iw/task_gui.py:80
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""

#: ../iw/task_gui.py:90
#, python-format
msgid ""
"Unable to find a group file for %s.  This will prevent manual selection of "
"packages from the repository from working"
msgstr ""

#: ../iw/task_gui.py:122
msgid "Invalid Repository Name"
msgstr ""

#: ../iw/task_gui.py:123
msgid "You must provide a repository name."
msgstr ""

#: ../iw/task_gui.py:131
msgid "Invalid Repository URL"
msgstr ""

#: ../iw/task_gui.py:132
msgid "You must provide an HTTP or FTP URL to a repository."
msgstr ""

#: ../iw/task_gui.py:146
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""

#: ../iw/timezone_gui.py:61 ../textw/timezone_text.py:89
msgid "Time Zone Selection"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:27 ../textw/upgrade_bootloader_text.py:72
msgid "Upgrade Boot Loader Configuration"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:66
msgid "_Update boot loader configuration"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:67
msgid "This will update your current boot loader."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:70 ../textw/upgrade_bootloader_text.py:49
#, python-format
msgid ""
"The installer has detected the %s boot loader currently installed on %s."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:74
msgid "This is the recommended option."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:78 ../textw/upgrade_bootloader_text.py:57
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:87
msgid "_Create new boot loader configuration"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:89
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:96
msgid "_Skip boot loader updating"
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:97
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""

#: ../iw/upgrade_bootloader_gui.py:109
msgid "What would you like to do?"
msgstr ""

#: ../iw/upgrade_migratefs_gui.py:28 ../textw/upgrade_text.py:31
msgid "Migrate File Systems"
msgstr ""

#: ../iw/upgrade_migratefs_gui.py:52 ../textw/upgrade_text.py:33
#, python-format
msgid ""
"This release of %s supports the ext3 journaling file system, which has "
"several benefits over the ext2 file system traditionally shipped in %s.  "
"This installation program can migrate the ext2 formatted partitions to ext3 "
"without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: ../iw/upgrade_swap_gui.py:29
msgid "Upgrade Swap Partition"
msgstr ""

#: ../iw/upgrade_swap_gui.py:89 ../textw/upgrade_text.py:89
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have %"
"dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""

#: ../iw/upgrade_swap_gui.py:96
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""

#: ../iw/upgrade_swap_gui.py:108
msgid "I _want to create a swap file"
msgstr ""

#: ../iw/upgrade_swap_gui.py:117
msgid "Select the _partition to put the swap file on:"
msgstr ""

#: ../iw/upgrade_swap_gui.py:136 ../textw/upgrade_text.py:107
msgid "Partition"
msgstr ""

#: ../iw/upgrade_swap_gui.py:136
msgid "Free Space (MB)"
msgstr ""

#: ../iw/upgrade_swap_gui.py:154
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""

#: ../iw/upgrade_swap_gui.py:169
msgid "Swap file _size (MB):"
msgstr ""

#: ../iw/upgrade_swap_gui.py:179
msgid "I _don't want to create a swap file"
msgstr ""

#: ../iw/upgrade_swap_gui.py:189
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""

#: ../iw/upgrade_swap_gui.py:197 ../textw/upgrade_text.py:173
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr ""

#: ../iw/upgrade_swap_gui.py:204 ../textw/upgrade_text.py:168
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr ""

#: ../iw/zipl_gui.py:28
msgid "z/IPL Boot Loader Configuration"
msgstr ""

#: ../iw/zipl_gui.py:52
msgid "The z/IPL boot loader will be installed on your system."
msgstr ""

#: ../iw/zipl_gui.py:54
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""

#: ../iw/zipl_gui.py:81 ../textw/zipl_text.py:62
msgid "Kernel Parameters"
msgstr ""

#: ../iw/zipl_gui.py:84 ../iw/zipl_gui.py:87
msgid "Chandev Parameters"
msgstr ""

#: ../textw/bootloader_text.py:28
msgid "Which boot loader would you like to use?"
msgstr ""

#: ../textw/bootloader_text.py:38
msgid "Use GRUB Boot Loader"
msgstr ""

#: ../textw/bootloader_text.py:39
msgid "No Boot Loader"
msgstr ""

#: ../textw/bootloader_text.py:59
msgid "Skip Boot Loader"
msgstr ""

#: ../textw/bootloader_text.py:60
msgid ""
"You have elected not to install any boot loader, which is not recommended "
"unless you have an advanced need. Booting your system into Linux directly "
"from the hard drive almost always requires a boot loader.\n"
"\n"
"Are you sure you want to skip boot loader installation?"
msgstr ""

#: ../textw/bootloader_text.py:92
msgid ""
"A few systems need to pass special options to the kernel at boot time to "
"function properly. If you need to pass boot options to the kernel, enter "
"them now. If you don't need any or aren't sure, leave this blank."
msgstr ""

#: ../textw/bootloader_text.py:101
msgid "Force use of LBA32 (not normally required)"
msgstr ""

#: ../textw/bootloader_text.py:165
msgid "Where do you want to install the boot loader?"
msgstr ""

#: ../textw/bootloader_text.py:193 ../textw/bootloader_text.py:258
msgid "Boot label"
msgstr ""

#: ../textw/bootloader_text.py:197
msgid "Clear"
msgstr ""

#: ../textw/bootloader_text.py:205
msgid "Edit Boot Label"
msgstr ""

#: ../textw/bootloader_text.py:223 ../textw/bootloader_text.py:228
msgid "Invalid Boot Label"
msgstr ""

#: ../textw/bootloader_text.py:224
msgid "Boot label may not be empty."
msgstr ""

#: ../textw/bootloader_text.py:229
msgid "Boot label contains illegal characters."
msgstr ""

#: ../textw/bootloader_text.py:273 ../textw/constants_text.py:56
#: ../textw/partition_text.py:1452
msgid "Edit"
msgstr ""

#: ../textw/bootloader_text.py:277
#, python-format
msgid ""
"The boot manager %s uses can boot other operating systems as well.  Please "
"tell me what partitions you would like to be able to boot and what label you "
"want to use for each of them."
msgstr ""

#: ../textw/bootloader_text.py:290
msgid ""
" <Space> select | <F2> select default | <F4> delete | <F12> next screen>"
msgstr ""

#: ../textw/bootloader_text.py:386
msgid ""
"A boot loader password prevents users from passing arbitrary options to the "
"kernel.  For highest security, you should set a password, but a password is "
"not necessary for more casual users."
msgstr ""

#: ../textw/bootloader_text.py:396
msgid "Use a GRUB Password"
msgstr ""

#: ../textw/bootloader_text.py:408
msgid "Boot Loader Password:"
msgstr ""

#: ../textw/bootloader_text.py:409
msgid "Confirm:"
msgstr ""

#: ../textw/bootloader_text.py:438
msgid "Passwords Do Not Match"
msgstr ""

#: ../textw/bootloader_text.py:443
msgid "Password Too Short"
msgstr ""

#: ../textw/bootloader_text.py:444
msgid "Boot loader password is too short"
msgstr ""

#: ../textw/complete_text.py:26
msgid ""
"Press <Enter> to end the installation process.\n"
"\n"
msgstr ""

#: ../textw/complete_text.py:27
msgid "<Enter> to exit"
msgstr ""

#: ../textw/complete_text.py:31
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"%s%s"
msgstr ""

#: ../textw/complete_text.py:34
#, python-format
msgid ""
"For information on errata (updates and bug fixes), visit http://www.redhat."
"com/errata/.\n"
"\n"
"Information on using your system is available in the %s manuals at http://"
"www.redhat.com/docs/."
msgstr ""

#: ../textw/complete_text.py:40
msgid "Complete"
msgstr ""

#: ../textw/confirm_text.py:22
msgid "Installation to begin"
msgstr ""

#: ../textw/confirm_text.py:23
#, python-format
msgid ""
"A complete log of your installation will be in %s after rebooting your "
"system. You may want to keep this file for later reference."
msgstr ""

#: ../textw/confirm_text.py:26 ../textw/confirm_text.py:28
#: ../textw/confirm_text.py:54 ../textw/confirm_text.py:56
#: ../textw/constants_text.py:44 ../loader2/cdinstall.c:392
#: ../loader2/driverdisk.c:274 ../loader2/driverdisk.c:305
#: ../loader2/driverdisk.c:390 ../loader2/driverselect.c:70
#: ../loader2/driverselect.c:182 ../loader2/driverselect.c:207
#: ../loader2/hdinstall.c:265 ../loader2/hdinstall.c:320 ../loader2/kbd.c:131
#: ../loader2/loader.c:338 ../loader2/loader.c:1002 ../loader2/loader.c:1025
#: ../loader2/net.c:268 ../loader2/net.c:307 ../loader2/net.c:739
#: ../loader2/net.c:1064 ../loader2/net.c:1820 ../loader2/nfsinstall.c:57
#: ../loader2/urls.c:282 ../loader2/urls.c:470
msgid "Back"
msgstr ""

#: ../textw/confirm_text.py:50
msgid "Upgrade to begin"
msgstr ""

#: ../textw/confirm_text.py:51
#, python-format
msgid ""
"A complete log of your upgrade will be in %s after rebooting your system. "
"You may want to keep this file for later reference."
msgstr ""

#: ../textw/grpselect_text.py:87
msgid "Please select the package groups you would like to install."
msgstr ""

#: ../textw/grpselect_text.py:105
msgid ""
"<Space>,<+>,<-> selection   |   <F2> Group Details   |   <F12> next screen"
msgstr ""

#: ../textw/grpselect_text.py:117
msgid "No optional packages to select"
msgstr ""

#: ../textw/grpselect_text.py:139
msgid "Package Group Details"
msgstr ""

#: ../textw/keyboard_text.py:38
msgid "Keyboard Selection"
msgstr ""

#: ../textw/keyboard_text.py:39
msgid "Which model keyboard is attached to this computer?"
msgstr ""

#: ../textw/mouse_text.py:39
msgid "What device is your mouse located on?"
msgstr ""

#: ../textw/mouse_text.py:71
msgid "Which model mouse is attached to this computer?"
msgstr ""

#: ../textw/mouse_text.py:82
msgid "Emulate 3 Buttons?"
msgstr ""

#: ../textw/mouse_text.py:85
msgid "Mouse Selection"
msgstr ""

#: ../textw/network_text.py:50
#, python-format
msgid ""
"You have not specified the field %s.  Depending on your network environment "
"this may cause problems later."
msgstr ""

#: ../textw/network_text.py:67
#, python-format
msgid "IPv%d prefix must be between 0 and %d."
msgstr ""

#: ../textw/network_text.py:71
msgid "Integer Required for Prefix"
msgstr ""

#: ../textw/network_text.py:72
#, python-format
msgid ""
"You must enter a valid integer for the %s.  For IPv4, the value can be "
"between 0 and 32.  For IPv6 it can be between 0 and 128."
msgstr ""

#: ../textw/network_text.py:79 tmp/netpostconfig.glade.h:16
msgid "Prefix (Netmask)"
msgstr ""

#: ../textw/network_text.py:81 tmp/netpostconfig.glade.h:15
msgid "Prefix"
msgstr ""

#: ../textw/network_text.py:134
msgid "Activate on boot"
msgstr ""

#: ../textw/network_text.py:141 ../loader2/net.c:719
#: tmp/netpostconfig.glade.h:9
msgid "Enable IPv4 support"
msgstr ""

#: ../textw/network_text.py:148 ../loader2/net.c:732
#: tmp/netpostconfig.glade.h:10
msgid "Enable IPv6 support"
msgstr ""

#: ../textw/network_text.py:161
msgid "P-to-P:"
msgstr ""

#: ../textw/network_text.py:179 tmp/netpostconfig.glade.h:7
#, fuzzy
msgid "ESSID:"
msgstr "_ESSID"

#: ../textw/network_text.py:188
msgid "WEP Key:"
msgstr ""

#: ../textw/network_text.py:201
#, python-format
msgid "Network Configuration for %s"
msgstr ""

#: ../textw/network_text.py:238 ../textw/network_text.py:241
msgid "point-to-point IP address"
msgstr ""

#: ../textw/network_text.py:264 tmp/netpostconfig.glade.h:5
#, fuzzy
msgid "Dynamic IP configuration (DHCP)"
msgstr "Configuración do Rato"

#: ../textw/network_text.py:267 ../textw/network_text.py:398
#, fuzzy
msgid "Manual address configuration"
msgstr "Configuración do Rato"

#: ../textw/network_text.py:287
#, fuzzy, python-format
msgid "IPv4 Configuration for %s"
msgstr "Configuración de VNC"

#: ../textw/network_text.py:317 ../textw/network_text.py:329
#: ../textw/network_text.py:332
msgid "IPv4 address"
msgstr ""

#: ../textw/network_text.py:321
#, fuzzy
msgid "IPv4 network mask"
msgstr "IP/Máscara de rede"

#: ../textw/network_text.py:347 ../textw/network_text.py:350
#: ../textw/network_text.py:353
msgid "IPv4 prefix (network mask)"
msgstr ""

#: ../textw/network_text.py:392 tmp/netpostconfig.glade.h:4
msgid "Automatic neighbor discovery"
msgstr ""

#: ../textw/network_text.py:395 tmp/netpostconfig.glade.h:6
#, fuzzy
msgid "Dynamic IP configuration (DHCPv6)"
msgstr "Configuración do Rato"

#: ../textw/network_text.py:422
#, fuzzy, python-format
msgid "IPv6 Configuration for %s"
msgstr "Configuración de VNC"

#: ../textw/network_text.py:452 ../textw/network_text.py:463
#: ../textw/network_text.py:466
msgid "IPv6 address"
msgstr ""

#: ../textw/network_text.py:456 ../textw/network_text.py:475
msgid "IPv6 prefix"
msgstr ""

#: ../textw/network_text.py:501
msgid "Configure Network Interface"
msgstr ""

#: ../textw/network_text.py:502
#, python-format
msgid "Would you like to configure the %s network interface in your system?"
msgstr ""

#: ../textw/network_text.py:516 ../textw/network_text.py:518
msgid "UNCONFIGURED"
msgstr ""

#: ../textw/network_text.py:527
msgid ""
"The current configuration settings for each interface are listed next to the "
"device name.  Unconfigured interfaces are shown as UNCONFIGURED.  To "
"configure an interface, highlight it and choose Edit.  When you are "
"finished, press OK to continue."
msgstr ""

#: ../textw/network_text.py:614
#, fuzzy
msgid "Active on boot"
msgstr "Activar ó Arrincar"

#: ../textw/network_text.py:616
#, fuzzy
msgid "Inactive on boot"
msgstr "Activar ó Arrincar"

#: ../textw/network_text.py:619
msgid "DHCP"
msgstr ""

#: ../textw/network_text.py:624
msgid "Auto IPv6"
msgstr ""

#: ../textw/network_text.py:626
msgid "DHCPv6"
msgstr ""

#: ../textw/network_text.py:632
#, python-format
msgid "%s, %s, %s"
msgstr ""

#: ../textw/network_text.py:634 ../textw/network_text.py:636
#, python-format
msgid "%s, %s"
msgstr ""

#: ../textw/network_text.py:656 ../loader2/net.c:1025
msgid "Gateway:"
msgstr ""

#: ../textw/network_text.py:665
msgid "Primary DNS:"
msgstr ""

#: ../textw/network_text.py:670
msgid "Secondary DNS:"
msgstr ""

#: ../textw/network_text.py:677
msgid "Miscellaneous Network Settings"
msgstr ""

#: ../textw/network_text.py:694 ../textw/network_text.py:697
#, fuzzy
msgid "gateway"
msgstr "Pasarela"

#: ../textw/network_text.py:704 ../textw/network_text.py:707
msgid "primary DNS"
msgstr ""

#: ../textw/network_text.py:716
msgid "secondary DNS"
msgstr ""

#: ../textw/network_text.py:750
msgid "automatically via DHCP"
msgstr ""

#: ../textw/network_text.py:753
msgid "manually"
msgstr ""

#: ../textw/network_text.py:772
msgid "Hostname Configuration"
msgstr ""

#: ../textw/network_text.py:775
msgid ""
"If your system is part of a larger network where hostnames are assigned by "
"DHCP, select automatically via DHCP. Otherwise, select manually and enter a "
"hostname for your system. If you do not, your system will be known as "
"'localhost.'"
msgstr ""

#: ../textw/network_text.py:801 ../textw/network_text.py:807
msgid "Invalid Hostname"
msgstr ""

#: ../textw/network_text.py:802
msgid "You have not specified a hostname."
msgstr ""

#: ../textw/partition_text.py:43
msgid "Must specify a value"
msgstr ""

#: ../textw/partition_text.py:46
msgid "Requested value is not an integer"
msgstr ""

#: ../textw/partition_text.py:48
msgid "Requested value is too large"
msgstr ""

#: ../textw/partition_text.py:102
#, python-format
msgid "RAID Device %s"
msgstr ""

#: ../textw/partition_text.py:232
#, python-format
msgid "Warning: %s"
msgstr ""

#: ../textw/partition_text.py:233
msgid "Modify Partition"
msgstr ""

#: ../textw/partition_text.py:233
msgid "Add anyway"
msgstr ""

#: ../textw/partition_text.py:271
msgid "Mount Point:"
msgstr ""

#: ../textw/partition_text.py:323
msgid "File System type:"
msgstr ""

#: ../textw/partition_text.py:357
msgid "Allowable Drives:"
msgstr ""

#: ../textw/partition_text.py:413
msgid "Fixed Size:"
msgstr ""

#: ../textw/partition_text.py:415
msgid "Fill maximum size of (MB):"
msgstr ""

#: ../textw/partition_text.py:419
msgid "Fill all available space:"
msgstr ""

#: ../textw/partition_text.py:442
msgid "Start Cylinder:"
msgstr ""

#: ../textw/partition_text.py:455
msgid "End Cylinder:"
msgstr ""

#: ../textw/partition_text.py:478
msgid "Volume Group:"
msgstr ""

#: ../textw/partition_text.py:500
msgid "RAID Level:"
msgstr ""

#: ../textw/partition_text.py:518
msgid "RAID Members:"
msgstr ""

#: ../textw/partition_text.py:537
msgid "Number of spares?"
msgstr ""

#: ../textw/partition_text.py:551
msgid "File System Type:"
msgstr ""

#: ../textw/partition_text.py:564
msgid "File System Label:"
msgstr ""

#: ../textw/partition_text.py:575
msgid "File System Option:"
msgstr ""

#: ../textw/partition_text.py:578 ../textw/partition_text.py:816
#: ../textw/partition_text.py:1053 ../textw/partition_text.py:1224
#, python-format
msgid "Format as %s"
msgstr ""

#: ../textw/partition_text.py:580 ../textw/partition_text.py:818
#: ../textw/partition_text.py:1055 ../textw/partition_text.py:1226
#, python-format
msgid "Migrate to %s"
msgstr ""

#: ../textw/partition_text.py:582 ../textw/partition_text.py:820
#: ../textw/partition_text.py:1057 ../textw/partition_text.py:1228
msgid "Leave unchanged"
msgstr ""

#: ../textw/partition_text.py:598 ../textw/partition_text.py:793
#: ../textw/partition_text.py:1033 ../textw/partition_text.py:1204
msgid "File System Options"
msgstr ""

#: ../textw/partition_text.py:601
msgid ""
"Please choose how you would like to prepare the file system on this "
"partition."
msgstr ""

#: ../textw/partition_text.py:609
msgid "Check for bad blocks"
msgstr ""

#: ../textw/partition_text.py:613
msgid "Leave unchanged (preserve data)"
msgstr ""

#: ../textw/partition_text.py:622
msgid "Format as:"
msgstr ""

#: ../textw/partition_text.py:642
msgid "Migrate to:"
msgstr ""

#: ../textw/partition_text.py:754
msgid "Force to be a primary partition"
msgstr ""

#: ../textw/partition_text.py:771
msgid "Not Supported"
msgstr ""

#: ../textw/partition_text.py:772
msgid "You can only edit LVM Volume Groups in the graphical installer."
msgstr ""

#: ../textw/partition_text.py:848 ../textw/partition_text.py:901
msgid "Invalid Entry for Partition Size"
msgstr ""

#: ../textw/partition_text.py:860
msgid "Invalid Entry for Maximum Size"
msgstr ""

#: ../textw/partition_text.py:879
msgid "Invalid Entry for Starting Cylinder"
msgstr ""

#: ../textw/partition_text.py:893
msgid "Invalid Entry for End Cylinder"
msgstr ""

#: ../textw/partition_text.py:1006
msgid "No RAID partitions"
msgstr ""

#: ../textw/partition_text.py:1007
msgid "At least two software RAID partitions are needed."
msgstr ""

#: ../textw/partition_text.py:1019 ../textw/partition_text.py:1191
msgid "Format partition?"
msgstr ""

#: ../textw/partition_text.py:1081
msgid "Invalid Entry for RAID Spares"
msgstr ""

#: ../textw/partition_text.py:1095
msgid "Too many spares"
msgstr ""

#: ../textw/partition_text.py:1096
msgid "You may not use any spares with a RAID0 array."
msgstr ""

#: ../textw/partition_text.py:1177
msgid "No Volume Groups"
msgstr ""

#: ../textw/partition_text.py:1178
msgid "No volume groups exist in which to create a logical volume"
msgstr ""

#: ../textw/partition_text.py:1302
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the maximum logical "
"volume size (%10.2f MB). "
msgstr ""

#: ../textw/partition_text.py:1321
#, python-format
msgid ""
"The current requested size (%10.2f MB) is larger than the available size in "
"the volume group (%10.2f MB)."
msgstr ""

#: ../textw/partition_text.py:1375
msgid "New Partition or Logical Volume?"
msgstr ""

#: ../textw/partition_text.py:1376
msgid "Would you like to create a new partition or a new logical volume?"
msgstr ""

#: ../textw/partition_text.py:1378
msgid "partition"
msgstr ""

#: ../textw/partition_text.py:1378
msgid "logical volume"
msgstr ""

#: ../textw/partition_text.py:1451
msgid "New"
msgstr ""

#: ../textw/partition_text.py:1453
msgid "Delete"
msgstr ""

#: ../textw/partition_text.py:1454
msgid "RAID"
msgstr ""

#: ../textw/partition_text.py:1457
msgid ""
"    F1-Help     F2-New      F3-Edit   F4-Delete    F5-Reset    F12-OK        "
msgstr ""

#: ../textw/partition_text.py:1489
msgid "No Root Partition"
msgstr ""

#: ../textw/partition_text.py:1490
msgid "Installation requires a / partition."
msgstr ""

#: ../textw/partition_text.py:1532
msgid "Partitioning Type"
msgstr ""

#: ../textw/partition_text.py:1534
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is reasonable for most users.  You can either choose to use this or create "
"your own."
msgstr ""

#: ../textw/partition_text.py:1558
msgid "Which drive(s) do you want to use for this installation?"
msgstr ""

#: ../textw/partition_text.py:1573
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr ""

#: ../textw/partition_text.py:1635
msgid "Review Partition Layout"
msgstr ""

#: ../textw/partition_text.py:1636
msgid "Review and modify partitioning layout?"
msgstr ""

#: ../textw/partition_text.py:1657 tmp/adddrive.glade.h:3
msgid "Advanced Storage Options"
msgstr ""

#: ../textw/partition_text.py:1658 tmp/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr ""

#: ../textw/partition_text.py:1681
#, fuzzy
msgid "Add FCP Device"
msgstr "Dispositivo RAID"

#: ../textw/partition_text.py:1682 tmp/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""

#: ../textw/partition_text.py:1705 tmp/iscsi-config.glade.h:5
msgid "Configure iSCSI Parameters"
msgstr ""

#: ../textw/partition_text.py:1706 tmp/iscsi-config.glade.h:6
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: ../textw/partition_text.py:1707
msgid "Target IP Address"
msgstr ""

#: ../textw/partition_text.py:1708
msgid "iSCSI Initiator Name"
msgstr ""

#: ../textw/partmethod_text.py:26
msgid "Autopartition"
msgstr ""

#: ../textw/partmethod_text.py:27
msgid "Disk Druid"
msgstr ""

#: ../textw/progress_text.py:51
msgid "Package Installation"
msgstr ""

#: ../textw/task_text.py:42
msgid "Package selection"
msgstr ""

#: ../textw/task_text.py:48 tmp/tasksel.glade.h:4
#, no-c-format, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""

#: ../textw/task_text.py:63
msgid "Customize software selection"
msgstr ""

#: ../textw/timezone_text.py:68
msgid "In which time zone are you located?"
msgstr ""

#: ../textw/timezone_text.py:86
msgid "System clock uses UTC"
msgstr ""

#: ../textw/upgrade_bootloader_text.py:53
#: ../textw/upgrade_bootloader_text.py:60
msgid "Update boot loader configuration"
msgstr ""

#: ../textw/upgrade_bootloader_text.py:64
msgid "Skip boot loader updating"
msgstr ""

#: ../textw/upgrade_bootloader_text.py:66
msgid "Create new boot loader configuration"
msgstr ""

#: ../textw/upgrade_text.py:107
msgid "Free Space"
msgstr ""

#: ../textw/upgrade_text.py:122
msgid "RAM detected (MB):"
msgstr ""

#: ../textw/upgrade_text.py:125
msgid "Suggested size (MB):"
msgstr ""

#: ../textw/upgrade_text.py:128
msgid "Swap file size (MB):"
msgstr ""

#: ../textw/upgrade_text.py:136
msgid "Add Swap"
msgstr ""

#: ../textw/upgrade_text.py:161
msgid "The value you entered is not a valid number."
msgstr ""

#: ../textw/upgrade_text.py:194
msgid "Reinstall System"
msgstr ""

#: ../textw/upgrade_text.py:203
msgid "System to Upgrade"
msgstr ""

#: ../textw/upgrade_text.py:204
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""

#: ../textw/userauth_text.py:27
msgid "Root Password"
msgstr ""

#: ../textw/userauth_text.py:29
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. Remember that the root password is a critical "
"part of system security!"
msgstr ""

#: ../textw/userauth_text.py:60
msgid "The root password must be at least 6 characters long."
msgstr ""

#: ../textw/welcome_text.py:23
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""

#: ../textw/zipl_text.py:26
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""

#: ../textw/zipl_text.py:58
msgid "z/IPL Configuration"
msgstr ""

#: ../textw/zipl_text.py:66 ../textw/zipl_text.py:70
msgid "Chandev line "
msgstr ""

#: ../installclasses/fedora.py:15
msgid "_Fedora"
msgstr ""

#: ../installclasses/fedora.py:16 ../installclasses/rhel.py:22
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. What additional tasks would you like your system to "
"include support for?"
msgstr ""

#: ../installclasses/fedora.py:25
msgid "Office and Productivity"
msgstr ""

#: ../installclasses/fedora.py:26 ../installclasses/rhel.py:35
#: ../installclasses/rhel.py:40
msgid "Software Development"
msgstr ""

#: ../installclasses/fedora.py:27 ../installclasses/rhel.py:39
msgid "Web server"
msgstr ""

#: ../installclasses/rhel.py:21
msgid "Red Hat Enterprise Linux"
msgstr ""

#: ../installclasses/rhel.py:32
msgid "Office"
msgstr ""

#: ../installclasses/rhel.py:33
msgid "Multimedia"
msgstr ""

#: ../installclasses/rhel.py:44
#, fuzzy
msgid "Virtualization"
msgstr "Noraboa"

#: ../installclasses/rhel.py:45
msgid "Clustering"
msgstr ""

#: ../installclasses/rhel.py:46
msgid "Storage Clustering"
msgstr ""

#: ../installclasses/rhel.py:50
#, fuzzy
msgid "Installation Number"
msgstr "Progreso da Instalación"

#: ../installclasses/rhel.py:51
msgid ""
"To install the full set of supported packages included in your subscription, "
"please enter your Installation Number"
msgstr ""

#: ../installclasses/rhel.py:54
msgid ""
"If you're unable to locate the Installation Number, consult http://www."
"redhat.com/apps/support/in.html.\n"
"\n"
"If you skip:\n"
"* You may not get access to the full set of packages included in your "
"subscription.\n"
"* It may result in an unsupported/uncertified installation of Red Hat "
"Enterprise Linux.\n"
"* You will not get software and security updates for packages not included "
"in your subscription."
msgstr ""

#: ../loader2/cdinstall.c:92 ../loader2/cdinstall.c:113
#: ../loader2/mediacheck.c:346
msgid "Media Check"
msgstr ""

#: ../loader2/cdinstall.c:92 ../loader2/cdinstall.c:95
#: ../loader2/cdinstall.c:113 ../loader2/cdinstall.c:121
#: ../loader2/method.c:421
msgid "Test"
msgstr ""

#: ../loader2/cdinstall.c:92 ../loader2/cdinstall.c:96
msgid "Eject CD"
msgstr ""

#: ../loader2/cdinstall.c:93
#, c-format
msgid ""
"Choose \"%s\" to test the CD currently in the drive, or \"%s\" to eject the "
"CD and insert another for testing."
msgstr ""

#: ../loader2/cdinstall.c:114
#, c-format
msgid ""
"If you would like to test additional media, insert the next CD and press \"%s"
"\". Testing each CD is not strictly required, however it is highly "
"recommended.  Minimally, the CDs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each CD prior to using it again."
msgstr ""

#: ../loader2/cdinstall.c:137 ../loader2/cdinstall.c:384
#, c-format
msgid ""
"The %s CD was not found in any of your CDROM drives. Please insert the %s CD "
"and press %s to retry."
msgstr ""

#: ../loader2/cdinstall.c:258
msgid "CD Found"
msgstr ""

#: ../loader2/cdinstall.c:259
#, c-format
msgid ""
"To begin testing the CD media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""

#: ../loader2/cdinstall.c:379
#, c-format
msgid ""
"No %s CD was found which matches your boot media.  Please insert the %s CD "
"and press %s to retry."
msgstr ""

#: ../loader2/cdinstall.c:391
msgid "CD Not Found"
msgstr ""

#: ../loader2/cdinstall.c:464
msgid "Cannot find kickstart file on CDROM."
msgstr ""

#: ../loader2/driverdisk.c:128 ../loader2/firewire.c:55
msgid "Loading"
msgstr ""

#: ../loader2/driverdisk.c:128
msgid "Reading driver disk..."
msgstr ""

#: ../loader2/driverdisk.c:268 ../loader2/driverdisk.c:300
msgid "Driver Disk Source"
msgstr ""

#: ../loader2/driverdisk.c:269
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""

#: ../loader2/driverdisk.c:301
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""

#: ../loader2/driverdisk.c:339
msgid "Failed to mount partition."
msgstr ""

#: ../loader2/driverdisk.c:347
msgid "Select driver disk image"
msgstr ""

#: ../loader2/driverdisk.c:348
msgid "Select the file which is your driver disk image."
msgstr ""

#: ../loader2/driverdisk.c:377
msgid "Failed to load driver disk from file."
msgstr ""

#: ../loader2/driverdisk.c:388
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""

#: ../loader2/driverdisk.c:390
msgid "Insert Driver Disk"
msgstr ""

#: ../loader2/driverdisk.c:405
msgid "Failed to mount driver disk."
msgstr ""

#: ../loader2/driverdisk.c:415
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr ""

#: ../loader2/driverdisk.c:479
msgid "Manually choose"
msgstr ""

#: ../loader2/driverdisk.c:480
msgid "Load another disk"
msgstr ""

#: ../loader2/driverdisk.c:481
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""

#: ../loader2/driverdisk.c:520
msgid "Driver disk"
msgstr ""

#: ../loader2/driverdisk.c:521
msgid "Do you have a driver disk?"
msgstr ""

#: ../loader2/driverdisk.c:531
msgid "More Driver Disks?"
msgstr ""

#: ../loader2/driverdisk.c:532
msgid "Do you wish to load any more driver disks?"
msgstr ""

#: ../loader2/driverdisk.c:581 ../loader2/driverdisk.c:618
#: ../loader2/hdinstall.c:406 ../loader2/kickstart.c:127
#: ../loader2/kickstart.c:137 ../loader2/kickstart.c:180
#: ../loader2/kickstart.c:485 ../loader2/modules.c:1042
#: ../loader2/modules.c:1055 ../loader2/net.c:1611 ../loader2/net.c:1634
#: ../loader2/nfsinstall.c:251 ../loader2/urlinstall.c:456
#: ../loader2/urlinstall.c:465 ../loader2/urlinstall.c:476
msgid "Kickstart Error"
msgstr ""

#: ../loader2/driverdisk.c:582
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr ""

#: ../loader2/driverdisk.c:619
#, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s:%s"
msgstr ""

#: ../loader2/driverselect.c:60
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""

#: ../loader2/driverselect.c:80
msgid "Enter Module Parameters"
msgstr ""

#: ../loader2/driverselect.c:181
msgid "No drivers found"
msgstr ""

#: ../loader2/driverselect.c:181
msgid "Load driver disk"
msgstr ""

#: ../loader2/driverselect.c:182
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""

#: ../loader2/driverselect.c:200
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""

#: ../loader2/driverselect.c:208
msgid "Specify optional module arguments"
msgstr ""

#: ../loader2/driverselect.c:228
msgid "Select Device Driver to Load"
msgstr ""

#: ../loader2/firewire.c:55 ../loader2/windows.c:57
#, c-format
msgid "Loading %s driver..."
msgstr ""

#: ../loader2/hdinstall.c:102 ../loader2/nfsinstall.c:198
#: ../loader2/urlinstall.c:160
#, c-format
msgid ""
"The %s installation tree in that directory does not seem to match your boot "
"media."
msgstr ""

#: ../loader2/hdinstall.c:158
msgid ""
"An error occured reading the install from the ISO images. Please check your "
"ISO images and try again."
msgstr ""

#: ../loader2/hdinstall.c:266
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""

#: ../loader2/hdinstall.c:281
#, c-format
msgid ""
"What partition and directory on that partition hold the CD (iso9660) images "
"for %s? If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""

#: ../loader2/hdinstall.c:304
msgid "Directory holding images:"
msgstr ""

#: ../loader2/hdinstall.c:332
msgid "Select Partition"
msgstr ""

#: ../loader2/hdinstall.c:372
#, c-format
msgid "Device %s does not appear to contain %s CDROM images."
msgstr ""

#: ../loader2/hdinstall.c:407
#, c-format
msgid "Bad argument to HD kickstart method command %s: %s"
msgstr ""

#: ../loader2/hdinstall.c:477 ../loader2/hdinstall.c:533
msgid "Cannot find kickstart file on hard drive."
msgstr ""

#: ../loader2/hdinstall.c:520
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr ""

#: ../loader2/kbd.c:129
msgid "Keyboard Type"
msgstr ""

#: ../loader2/kbd.c:130
msgid "What type of keyboard do you have?"
msgstr ""

#: ../loader2/kickstart.c:128
#, c-format
msgid "Error opening kickstart file %s: %s"
msgstr ""

#: ../loader2/kickstart.c:138
#, c-format
msgid "Error reading contents of kickstart file %s: %s"
msgstr ""

#: ../loader2/kickstart.c:181
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr ""

#: ../loader2/kickstart.c:280
msgid "Cannot find ks.cfg on boot floppy."
msgstr ""

#: ../loader2/kickstart.c:344
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""

#: ../loader2/kickstart.c:353
msgid "Error downloading kickstart file"
msgstr ""

#: ../loader2/kickstart.c:486
#, c-format
msgid "Bad argument to shutdown kickstart method command %s: %s"
msgstr ""

#: ../loader2/lang.c:56
#, c-format
msgid "Welcome to %s - Rescue Mode"
msgstr ""

#: ../loader2/lang.c:57 ../loader2/loader.c:161
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""

#: ../loader2/lang.c:366
msgid "Choose a Language"
msgstr ""

#: ../loader2/loader.c:120
msgid "Local CDROM"
msgstr ""

#: ../loader2/loader.c:122
msgid "Hard drive"
msgstr ""

#: ../loader2/loader.c:123
msgid "NFS directory"
msgstr ""

#: ../loader2/loader.c:333
msgid "Update Disk Source"
msgstr ""

#: ../loader2/loader.c:334
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""

#: ../loader2/loader.c:349
#, c-format
msgid "Insert your updates disk into /dev/%s and press \"OK\" to continue."
msgstr ""

#: ../loader2/loader.c:351
msgid "Updates Disk"
msgstr ""

#: ../loader2/loader.c:363
msgid "Failed to mount updates disk"
msgstr ""

#: ../loader2/loader.c:366
msgid "Updates"
msgstr ""

#: ../loader2/loader.c:366
msgid "Reading anaconda updates..."
msgstr ""

#: ../loader2/loader.c:406
msgid ""
"No hard drives have been found.  You probably need to manually choose device "
"drivers for the installation to succeed.  Would you like to select drivers "
"now?"
msgstr ""

#: ../loader2/loader.c:835
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr ""

#: ../loader2/loader.c:994
msgid "Rescue Method"
msgstr ""

#: ../loader2/loader.c:995
msgid "Installation Method"
msgstr ""

#: ../loader2/loader.c:997
msgid "What type of media contains the rescue image?"
msgstr ""

#: ../loader2/loader.c:999
msgid "What type of media contains the packages to be installed?"
msgstr ""

#: ../loader2/loader.c:1024
msgid "No driver found"
msgstr ""

#: ../loader2/loader.c:1024
msgid "Select driver"
msgstr ""

#: ../loader2/loader.c:1025
msgid "Use a driver disk"
msgstr ""

#: ../loader2/loader.c:1026
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""

#: ../loader2/loader.c:1219
msgid "The following devices have been found on your system."
msgstr ""

#: ../loader2/loader.c:1221
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""

#: ../loader2/loader.c:1225
msgid "Devices"
msgstr ""

#: ../loader2/loader.c:1226
msgid "Done"
msgstr ""

#: ../loader2/loader.c:1227
msgid "Add Device"
msgstr ""

#: ../loader2/loader.c:1435
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr ""

#: ../loader2/loader.c:1822
#, c-format
msgid "Running anaconda, the %s rescue mode - please wait...\n"
msgstr ""

#: ../loader2/loader.c:1824
#, c-format
msgid "Running anaconda, the %s system installer - please wait...\n"
msgstr ""

#: ../loader2/mediacheck.c:330
msgid ""
"Unable to read the disc checksum from the primary volume descriptor.  This "
"probably means the disc was created without adding the checksum."
msgstr ""

#: ../loader2/mediacheck.c:338
#, c-format
msgid "Checking \"%s\"..."
msgstr ""

#: ../loader2/mediacheck.c:340
#, c-format
msgid "Checking media now..."
msgstr ""

#: ../loader2/mediacheck.c:387
#, c-format
msgid "Unable to find install image %s"
msgstr ""

#: ../loader2/mediacheck.c:397 ../loader2/mediacheck.c:414
msgid "FAILED"
msgstr ""

#: ../loader2/mediacheck.c:398
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""

#: ../loader2/mediacheck.c:408
msgid "PASSED"
msgstr ""

#: ../loader2/mediacheck.c:409
msgid "It is OK to install from this media."
msgstr ""

#: ../loader2/mediacheck.c:415
msgid "No checksum information available, unable to verify media."
msgstr ""

#: ../loader2/mediacheck.c:420
msgid "Media Check Result"
msgstr ""

#: ../loader2/mediacheck.c:424
#, c-format
msgid ""
"%s for the image:\n"
"\n"
"   %s"
msgstr ""

#: ../loader2/mediacheck.c:428
#, c-format
msgid ""
"The media check %s\n"
"\n"
"%s"
msgstr ""

#: ../loader2/method.c:157 ../loader2/method.c:375 ../loader2/method.c:460
#, c-format
msgid "Failed to read directory %s: %s"
msgstr ""

#: ../loader2/method.c:418
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""

#: ../loader2/method.c:421
msgid "Checksum Test"
msgstr ""

#: ../loader2/modules.c:1043
#, c-format
msgid "Bad argument to device kickstart method command %s: %s"
msgstr ""

#: ../loader2/modules.c:1056
msgid ""
"Both module type and name must be specified for the kickstart device command."
msgstr ""

#: ../loader2/net.c:59
#, c-format
msgid ""
"Please enter the following information:\n"
"\n"
"    o the name or IP number of your %s server\n"
"    o the directory on that server containing\n"
"      %s for your architecture\n"
msgstr ""

#: ../loader2/net.c:94
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""

#: ../loader2/net.c:257
#, c-format
msgid ""
"%s is a wireless network adapter.  Please provide the ESSID and encryption "
"key needed to access your wireless network.  If no key is needed, leave this "
"field blank and the install will continue."
msgstr ""

#: ../loader2/net.c:263
msgid "ESSID"
msgstr ""

#: ../loader2/net.c:264
msgid "Encryption Key"
msgstr ""

#: ../loader2/net.c:267
msgid "Wireless Settings"
msgstr ""

#: ../loader2/net.c:298
msgid "Nameserver IP"
msgstr ""

#: ../loader2/net.c:302
msgid "Missing Nameserver"
msgstr ""

#: ../loader2/net.c:303
msgid ""
"Your IP address request returned configuration information, but it did not "
"include a nameserver address. If you do not have this information, you can "
"leave the field blank and the install will continue."
msgstr ""

#: ../loader2/net.c:325
msgid "Invalid IP Information"
msgstr ""

#: ../loader2/net.c:326
msgid "You entered an invalid IP address."
msgstr ""

#: ../loader2/net.c:583 ../loader2/net.c:691
msgid "Network Error"
msgstr ""

#: ../loader2/net.c:584 ../loader2/net.c:692
msgid "There was an error configuring your network interface."
msgstr ""

#: ../loader2/net.c:765
msgid "Configure TCP/IP"
msgstr ""

#: ../loader2/net.c:792
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr ""

#: ../loader2/net.c:799
msgid "IPv4 Needed for NFS"
msgstr ""

#: ../loader2/net.c:800
msgid "NFS installation method requires IPv4 support."
msgstr ""

#: ../loader2/net.c:920
msgid "IPv4 address:"
msgstr ""

#: ../loader2/net.c:932 ../loader2/net.c:984 tmp/netconfig.glade.h:1
#: tmp/netpostconfig.glade.h:1
msgid "/"
msgstr ""

#: ../loader2/net.c:972
msgid "IPv6 address:"
msgstr ""

#: ../loader2/net.c:1033
msgid "Name Server:"
msgstr ""

#: ../loader2/net.c:1069
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""

#: ../loader2/net.c:1085
#, fuzzy
msgid "Manual TCP/IP Configuration"
msgstr "Configuración do Rato"

#: ../loader2/net.c:1206 ../loader2/net.c:1212
msgid "Missing Information"
msgstr ""

#: ../loader2/net.c:1207
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""

#: ../loader2/net.c:1213
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr ""

#: ../loader2/net.c:1519
msgid "Determining host name and domain..."
msgstr ""

#: ../loader2/net.c:1612
#, c-format
msgid "Bad argument to kickstart network command %s: %s"
msgstr ""

#: ../loader2/net.c:1635
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr ""

#: ../loader2/net.c:1816
msgid "Networking Device"
msgstr ""

#: ../loader2/net.c:1817
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""

#: ../loader2/nfsinstall.c:47
msgid "NFS server name:"
msgstr ""

#: ../loader2/nfsinstall.c:50 ../loader2/urls.c:313
#, c-format
msgid "%s directory:"
msgstr ""

#: ../loader2/nfsinstall.c:55
msgid "NFS"
msgstr ""

#: ../loader2/nfsinstall.c:56
msgid "NFS Setup"
msgstr ""

#: ../loader2/nfsinstall.c:126
msgid "Hostname specified with no DNS configured"
msgstr ""

#: ../loader2/nfsinstall.c:202
#, c-format
msgid "That directory does not seem to contain a %s installation tree."
msgstr ""

#: ../loader2/nfsinstall.c:214
msgid "That directory could not be mounted from the server."
msgstr ""

#: ../loader2/nfsinstall.c:252
#, c-format
msgid "Bad argument to NFS kickstart method command %s: %s"
msgstr ""

#: ../loader2/telnetd.c:83 ../loader2/telnetd.c:125
msgid "Telnet"
msgstr ""

#: ../loader2/telnetd.c:83
msgid "Waiting for telnet connection..."
msgstr ""

#: ../loader2/telnetd.c:125
msgid "Running anaconda via telnet..."
msgstr ""

#: ../loader2/urlinstall.c:80
#, c-format
msgid "Unable to retrieve %s://%s/%s/%s."
msgstr ""

#: ../loader2/urlinstall.c:152
msgid "Unable to retrieve the install image."
msgstr ""

#: ../loader2/urlinstall.c:285
msgid "Media Detected"
msgstr ""

#: ../loader2/urlinstall.c:286
msgid "Local installation media detected..."
msgstr ""

#: ../loader2/urlinstall.c:457
#, c-format
msgid "Bad argument to Url kickstart method command %s: %s"
msgstr ""

#: ../loader2/urlinstall.c:466
msgid "Must supply a --url argument to Url kickstart method."
msgstr ""

#: ../loader2/urlinstall.c:477
#, c-format
msgid "Unknown Url method %s"
msgstr ""

#: ../loader2/urls.c:219
msgid "Retrieving"
msgstr ""

#: ../loader2/urls.c:286
msgid "FTP"
msgstr ""

#: ../loader2/urls.c:291
msgid "Web"
msgstr ""

#: ../loader2/urls.c:308
msgid "FTP site name:"
msgstr ""

#: ../loader2/urls.c:309
msgid "Web site name:"
msgstr ""

#: ../loader2/urls.c:328
msgid "Use non-anonymous ftp"
msgstr ""

#: ../loader2/urls.c:337
msgid "FTP Setup"
msgstr ""

#: ../loader2/urls.c:338
msgid "HTTP Setup"
msgstr ""

#: ../loader2/urls.c:348
msgid "You must enter a server name."
msgstr ""

#: ../loader2/urls.c:353
msgid "You must enter a directory."
msgstr ""

#: ../loader2/urls.c:358
msgid "Unknown Host"
msgstr ""

#: ../loader2/urls.c:359
#, c-format
msgid "%s is not a valid hostname."
msgstr ""

#: ../loader2/urls.c:430
msgid ""
"If you are using non anonymous ftp, enter the account name and password you "
"wish to use below."
msgstr ""

#: ../loader2/urls.c:435
msgid ""
"If you are using a HTTP proxy server enter the name of the HTTP proxy server "
"to use."
msgstr ""

#: ../loader2/urls.c:457
msgid "Account name:"
msgstr ""

#: ../loader2/urls.c:480
msgid "Further FTP Setup"
msgstr ""

#: ../loader2/urls.c:483
msgid "Further HTTP Setup"
msgstr ""

#: ../loader2/windows.c:56
msgid "Loading SCSI driver"
msgstr ""

#: tmp/adddrive.glade.h:1
msgid "Add _ZFCP LUN"
msgstr ""

#: tmp/adddrive.glade.h:2
msgid "Add _iSCSI target"
msgstr ""

#: tmp/adddrive.glade.h:4
msgid "Disable _dmraid device"
msgstr ""

#: tmp/addrepo.glade.h:1
msgid "<b>Repository _URL:</b>"
msgstr ""

#: tmp/addrepo.glade.h:2
msgid "<b>Repository _name:</b>"
msgstr ""

#: tmp/addrepo.glade.h:3
msgid "Add Repository"
msgstr ""

#: tmp/addrepo.glade.h:5
#, no-c-format
msgid ""
"Please provide the location where your additional software can be installed "
"from.  Note that this must be a valid repository for %s."
msgstr ""

#: tmp/addrepo.glade.h:6
msgid "_Add repository"
msgstr ""

#: tmp/anaconda.glade.h:1
msgid "Reboo_t"
msgstr ""

#: tmp/anaconda.glade.h:3
msgid "_Next"
msgstr ""

#: tmp/anaconda.glade.h:4
msgid "_Release Notes"
msgstr ""

#: tmp/autopart.glade.h:1
msgid ""
"Installation requires partitioning of your hard drive.  By default, a "
"partitioning layout is chosen which is reasonable for most users.  You can "
"either choose to use this or create your own."
msgstr ""

#: tmp/autopart.glade.h:2
msgid "Re_view and modify partitioning layout"
msgstr ""

#: tmp/autopart.glade.h:3
msgid "What drive would you like to boot this installation from?"
msgstr ""

#: tmp/autopart.glade.h:4
#, fuzzy
msgid "_Advanced storage configuration"
msgstr "Configuración da Rede"

#: tmp/autopart.glade.h:5
msgid "_Select the drive(s) to use for this installation."
msgstr ""

#: tmp/exn.glade.h:1
msgid "Exception Info"
msgstr ""

#: tmp/exn.glade.h:2
msgid "_Exception details"
msgstr ""

#: tmp/instkey.glade.h:2
#, no-c-format
msgid "%(instkey)s:"
msgstr ""

#: tmp/iscsi-config.glade.h:1
#, fuzzy
msgid "<b>_Password:</b>"
msgstr "Contrasinal:"

#: tmp/iscsi-config.glade.h:2
msgid "<b>_Target IP Address:</b>"
msgstr ""

#: tmp/iscsi-config.glade.h:3
#, fuzzy
msgid "<b>_Username:</b>"
msgstr "Nome de usuario"

#: tmp/iscsi-config.glade.h:4
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr ""

#: tmp/netconfig.glade.h:2
#, fuzzy
msgid "<b>Gateway:</b>"
msgstr "Pasarela"

#: tmp/netconfig.glade.h:3
msgid "<b>IPv4 Address:</b>"
msgstr ""

#: tmp/netconfig.glade.h:4
msgid "<b>IPv6 Address:</b>"
msgstr ""

#: tmp/netconfig.glade.h:5
msgid "<b>Nameserver:</b>"
msgstr ""

#: tmp/netconfig.glade.h:6
msgid "<b>_Interface:</b>"
msgstr ""

#: tmp/netconfig.glade.h:7
msgid "Enable IPv_4 support"
msgstr ""

#: tmp/netconfig.glade.h:8
msgid "Enable IPv_6 support"
msgstr ""

#: tmp/netconfig.glade.h:9
msgid "Enable network interface"
msgstr ""

#: tmp/netconfig.glade.h:10
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""

#: tmp/netconfig.glade.h:11
msgid "Use _dynamic IP configuration (DHCP)"
msgstr ""

#: tmp/netpostconfig.glade.h:2
msgid "<b>Description Goes Here</b>"
msgstr ""

#: tmp/netpostconfig.glade.h:3
msgid "<b>Hardware address: DE:AD:00:BE:EF:00</b>"
msgstr ""

#: tmp/netpostconfig.glade.h:8
#, fuzzy
msgid "Edit Interface"
msgstr "Editar a Interface %s"

#: tmp/netpostconfig.glade.h:11
msgid "Encryption Key:"
msgstr ""

#: tmp/netpostconfig.glade.h:13
#, fuzzy
msgid "Manual configuration"
msgstr "Configuración do Rato"

#: tmp/netpostconfig.glade.h:14
msgid "Point to Point (IP):"
msgstr ""

#: tmp/tasksel.glade.h:1
msgid "Customize _later"
msgstr ""

#: tmp/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""

#: tmp/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""

#: tmp/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr ""

#: tmp/tasksel.glade.h:7
msgid "_Customize now"
msgstr ""

#: tmp/zfcp-config.glade.h:1
#, fuzzy
msgid "<b>Device number:</b>"
msgstr "Número de dispositivo"

#: tmp/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr ""

#: tmp/zfcp-config.glade.h:3
#, fuzzy
msgid "<b>WWPN:</b>"
msgstr "Contrasinal:"

#: tmp/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr ""

#. generated from lang-table
msgid "Afrikaans"
msgstr ""

#. generated from lang-table
msgid "Arabic"
msgstr ""

#. generated from lang-table
msgid "Assamese"
msgstr ""

#. generated from lang-table
msgid "Bengali"
msgstr ""

#. generated from lang-table
msgid "Bengali(India)"
msgstr ""

#. generated from lang-table
msgid "Bulgarian"
msgstr ""

#. generated from lang-table
msgid "Catalan"
msgstr ""

#. generated from lang-table
msgid "Chinese(Simplified)"
msgstr ""

#. generated from lang-table
msgid "Chinese(Traditional)"
msgstr ""

#. generated from lang-table
msgid "Croatian"
msgstr ""

#. generated from lang-table
msgid "Czech"
msgstr ""

#. generated from lang-table
msgid "Danish"
msgstr ""

#. generated from lang-table
msgid "Dutch"
msgstr ""

#. generated from lang-table
msgid "English"
msgstr ""

#. generated from lang-table
msgid "Estonian"
msgstr ""

#. generated from lang-table
msgid "Finnish"
msgstr ""

#. generated from lang-table
msgid "French"
msgstr ""

#. generated from lang-table
msgid "German"
msgstr ""

#. generated from lang-table
msgid "Greek"
msgstr ""

#. generated from lang-table
msgid "Gujarati"
msgstr ""

#. generated from lang-table
msgid "Hindi"
msgstr ""

#. generated from lang-table
msgid "Hungarian"
msgstr ""

#. generated from lang-table
msgid "Icelandic"
msgstr ""

#. generated from lang-table
msgid "Iloko"
msgstr ""

#. generated from lang-table
msgid "Indonesian"
msgstr ""

#. generated from lang-table
msgid "Italian"
msgstr ""

#. generated from lang-table
msgid "Japanese"
msgstr ""

#. generated from lang-table
msgid "Kannada"
msgstr ""

#. generated from lang-table
msgid "Korean"
msgstr ""

#. generated from lang-table
msgid "Macedonian"
msgstr ""

#. generated from lang-table
msgid "Malay"
msgstr ""

#. generated from lang-table
msgid "Malayalam"
msgstr ""

#. generated from lang-table
msgid "Marathi"
msgstr ""

#. generated from lang-table
msgid "Norwegian"
msgstr ""

#. generated from lang-table
msgid "Northern Sotho"
msgstr ""

#. generated from lang-table
msgid "Oriya"
msgstr ""

#. generated from lang-table
msgid "Persian"
msgstr ""

#. generated from lang-table
msgid "Polish"
msgstr ""

#. generated from lang-table
msgid "Portuguese"
msgstr ""

#. generated from lang-table
msgid "Portuguese(Brazilian)"
msgstr ""

#. generated from lang-table
msgid "Punjabi"
msgstr ""

#. generated from lang-table
msgid "Russian"
msgstr ""

#. generated from lang-table
msgid "Serbian"
msgstr ""

#. generated from lang-table
msgid "Serbian(Latin)"
msgstr ""

#. generated from lang-table
msgid "Sinhala"
msgstr ""

#. generated from lang-table
msgid "Slovak"
msgstr ""

#. generated from lang-table
msgid "Slovenian"
msgstr ""

#. generated from lang-table
msgid "Spanish"
msgstr ""

#. generated from lang-table
msgid "Swedish"
msgstr ""

#. generated from lang-table
msgid "Tamil"
msgstr ""

#. generated from lang-table
msgid "Telugu"
msgstr ""

#. generated from lang-table
msgid "Turkish"
msgstr ""

#. generated from lang-table
msgid "Ukrainian"
msgstr ""

#. generated from lang-table
msgid "Vietnamese"
msgstr ""

#. generated from lang-table
msgid "Welsh"
msgstr ""

#. generated from lang-table
msgid "Zulu"
msgstr ""

#~ msgid "Installing %s-%s-%s... "
#~ msgstr "Instalando %s-%s-%s... "

#~ msgid "_Format drive"
#~ msgstr "_Formatar unidade"

#~ msgid "Error mounting file system on %s: %s"
#~ msgstr "Erro ó montar o sistema de ficheiros en %s: %s"

#~ msgid "Upgrade Existing System"
#~ msgstr "Actualizar Sistema Existente"

#~ msgid "Upgrade"
#~ msgstr "Actualizar"

#~ msgid "WWPN"
#~ msgstr "WWPN"

#~ msgid "SCSI LUN"
#~ msgstr "SCSI LUN"

#~ msgid "_Activate on boot"
#~ msgstr "_Activar ó arrincar"

#~ msgid "Configure %s"
#~ msgstr "Configurar %s"

#~ msgid "Configure using _DHCP"
#~ msgstr "Configurar usando _DHCP"

#~ msgid "Net_mask"
#~ msgstr "_Máscara de rede"

#~ msgid "Installing %s-%s-%s.%s.\n"
#~ msgstr "Instalando %s-%s-%s.%s.\n"

#~ msgid "_Cancel"
#~ msgstr "_Cancelar"
